<!DOCTYPE html>
<!--[if lte IE 8 ]><html lang="en" class="no-js oldIE"><![endif]-->
<!--[if (gt IE 8)|!(IE)]><!--><html lang="en" class="no-js"><!--<![endif]-->
<head>
    <title>Dynamics of the quorum sensing switch: stochastic and non-stationary effects | BMC Systems Biology | Full Text</title>
    
    
    <meta name="journal_id" content='12918'/>
    
    <meta name="dc.identifier" content='10.1186/1752-0509-7-6'/>
    
    <meta name="DOI" content='10.1186/1752-0509-7-6'/>
    
    <meta name="dc.title" content='Dynamics of the quorum sensing switch: stochastic and non-stationary effects'/>
    
    <meta name="dc.description" content='A wide range of bacteria species are known to communicate through the so called quorum sensing (QS) mechanism by means of which they produce a small molecule that can freely diffuse in the environment and in the cells. Upon reaching a threshold concentration, the signalling molecule activates the QS-controlled genes that promote phenotypic changes. This mechanism, for its simplicity, has become the model system for studying the emergence of a global response in prokaryotic cells. Yet, how cells precisely measure the signal concentration and act coordinately, despite the presence of fluctuations that unavoidably affects cell regulation and signalling, remains unclear. We propose a model for the QS signalling mechanism in Vibrio fischeri based on the synthetic strains lux01 and lux02. Our approach takes into account the key regulatory interactions between LuxR and LuxI, the autoinducer transport, the cellular growth and the division dynamics. By using both deterministic and stochastic models, we analyze the response and dynamics at the single-cell level and compare them to the global response at the population level. Our results show how fluctuations interfere with the synchronization of the cell activation and lead to a bimodal phenotypic distribution. In this context, we introduce the concept of precision in order to characterize the reliability of the QS communication process in the colony. We show that increasing the noise in the expression of LuxR helps cells to get activated at lower autoinducer concentrations but, at the same time, slows down the global response. The precision of the QS switch under non-stationary conditions decreases with noise, while at steady-state it is independent of the noise value. Our in silico experiments show that the response of the LuxR/LuxI system depends on the interplay between non-stationary and stochastic effects and that the burst size of the transcription/translation noise at the level of LuxR controls the phenotypic variability of the population. These results, together with recent experimental evidences on LuxR regulation in wild-type species, suggest that bacteria have evolved mechanisms to regulate the intensity of those fluctuations.'/>
    
    <meta name="dc.source" content='BMC Systems Biology 2013 7:1'/>
    
    <meta name="dc.format" content='text/html'/>
    
    <meta name="dc.publisher" content='BioMed Central'/>
    
    <meta name="dc.date" content='2013-01-16'/>
    
    <meta name="dc.type" content='OriginalPaper'/>
    
    <meta name="dc.language" content='En'/>
    
    <meta name="dc.copyright" content='2013 Weber and Buceta; licensee BioMed Central Ltd.'/>
    
    <meta name="dc.rights" content='http://creativecommons.org/licenses/by/2.0'/>
    
    <meta name="dc.rightsAgent" content='reprints@biomedcentral.com'/>
    
    <meta name="prism.issn" content='1752-0509'/>
    
    <meta name="prism.doi" content='10.1186/1752-0509-7-6'/>
    
    <meta name="prism.publicationName" content='BMC Systems Biology'/>
    
    <meta name="prism.publicationDate" content='2013-01-16'/>
    
    <meta name="prism.volume" content='7'/>
    
    <meta name="prism.number" content='1'/>
    
    <meta name="prism.section" content='OriginalPaper'/>
    
    <meta name="prism.startingPage" content='6'/>
    
    <meta name="prism.copyright" content='2013 Weber and Buceta; licensee BioMed Central Ltd.'/>
    
    <meta name="prism.rightsAgent" content='reprints@biomedcentral.com'/>
    
    <meta name="prism.url" content='http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-6'/>
    
    <meta name="citation_pdf_url" content='http://bmcsystbiol.biomedcentral.com/track/pdf/10.1186/1752-0509-7-6?site=bmcsystbiol.biomedcentral.com'/>
    
    <meta name="citation_abstract_html_url" content='http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-6'/>
    
    <meta name="citation_fulltext_html_url" content='http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-6'/>
    
    <meta name="citation_journal_title" content='BMC Systems Biology'/>
    
    <meta name="citation_publisher" content='BioMed Central'/>
    
    <meta name="citation_issn" content='1752-0509'/>
    
    <meta name="citation_title" content='Dynamics of the quorum sensing switch: stochastic and non-stationary effects'/>
    
    <meta name="citation_volume" content='7'/>
    
    <meta name="citation_issue" content='1'/>
    
    <meta name="citation_date" content='2013/01/16'/>
    
    <meta name="citation_publication_date" content='2013/01/16'/>
    
    <meta name="citation_firstpage" content='6'/>
    
    <meta name="citation_doi" content='10.1186/1752-0509-7-6'/>
    
    <meta name="citation_fulltext_world_readable" content=''/>
    
    <meta name="dc.creator" content='Marc Weber'/>
    
    <meta name="dc.creator" content='Javier Buceta'/>
    
    <meta name="dc.subject" content='Bioinformatics'/>
    
    <meta name="dc.subject" content='Systems Biology'/>
    
    <meta name="dc.subject" content='Simulation and Modeling'/>
    
    <meta name="dc.subject" content='Computational Biology/Bioinformatics'/>
    
    <meta name="dc.subject" content='Physiological, Cellular and Medical Topics'/>
    
    <meta name="dc.subject" content='Algorithms'/>
    
    <meta name="citation_author" content='Marc Weber'/>
    
    <meta name="citation_author_institution" content='Computer Simulation and Modelling (Co.S.Mo.) Lab, Parc Cient&#237;fic de Barcelona, Barcelona, Spain'/>
    
    <meta name="citation_author" content='Javier Buceta'/>
    
    <meta name="citation_author_institution" content='Computer Simulation and Modelling (Co.S.Mo.) Lab, Parc Cient&#237;fic de Barcelona, Barcelona, Spain'/>
    
    <meta name="citation_reference" content='citation_journal_title=Cell; citation_title=Bacterially speaking; citation_author=BL Bassler; citation_author=R Losick; citation_volume=125; citation_issue=2; citation_publication_date=2006; citation_pages=237-46; citation_doi=10.1016/j.cell.2006.04.001; citation_id=CR1;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BMC Syst Biol; citation_title=Theoretical basis of the community effect in development; citation_author=Y Saka; citation_author=C Lhoussaine; citation_author=C Kuttler; citation_author=E Ullner; citation_author=M Thiel; citation_volume=5; citation_publication_date=2011; citation_pages=54; citation_doi=10.1186/1752-0509-5-54; citation_id=CR2;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Microbiol (Reading, England); citation_title=Quorum sensing in bacterial virulence; citation_author=LCM Antunes; citation_author=RBR Ferreira; citation_author=MMC Buckner; citation_author=BB Finlay; citation_volume=156; citation_issue=Pt 8; citation_publication_date=2010; citation_pages=2271-82; citation_doi=10.1099/mic.0.038794-0; citation_id=CR3;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Curr Opin Biotechnol; citation_title=Engineering multicellular systems by cell-cell communication; citation_author=A Pai; citation_author=Y Tanouchi; citation_author=CH Collins; citation_author=L You; citation_volume=20; citation_issue=4; citation_publication_date=2009; citation_pages=461-70; citation_doi=10.1016/j.copbio.2009.08.006; citation_id=CR4;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Mol Biol; citation_title=Voigt Ca: Environmentally controlled invasion of cancer cells by engineered bacteria; citation_author=JC Anderson; citation_author=EJ Clarke; citation_author=AP Arkin; citation_volume=355; citation_issue=4; citation_publication_date=2006; citation_pages=619-27; citation_doi=10.1016/j.jmb.2005.10.076; citation_id=CR5;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Annu Rev Genet; citation_title=Bacterial quorum-sensing network architectures; citation_author=WL Ng; citation_author=BL Bassler; citation_volume=43; citation_publication_date=2009; citation_pages=197-222; citation_doi=10.1146/annurev-genet-102108-134304; citation_id=CR6;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Bull Math Biol; citation_title=A mathematical model for quorum sensing in Pseudomonas aeruginosa; citation_author=JD Dockery; citation_author=JP Keener; citation_volume=63; citation_publication_date=2001; citation_pages=95-116; citation_doi=10.1006/bulm.2000.0205; citation_id=CR7;'/>
    
    <meta name="citation_reference" content='citation_journal_title=OMICS; citation_title=Analysis of noise in quorum sensing; citation_author=C Cox; citation_author=G Peterson; citation_author=M Allen; citation_author=J Lancaster; citation_author=J McCollum; citation_author=D Austin; citation_author=L Yan; citation_author=G Sayler; citation_author=M Simpson; citation_volume=7; citation_issue=3; citation_publication_date=2003; citation_pages=317-334; citation_doi=10.1089/153623103322452422; citation_id=CR8;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Bio Syst; citation_title=Systems analysis of a quorum sensing network: design constraints imposed by the functional requirements, network topology and kinetic constants; citation_author=AB Goryachev; citation_author=DJ Toh; citation_author=T Lee; citation_volume=83; citation_issue=2-3; citation_publication_date=2006; citation_pages=178-87; citation_id=CR9;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Syst Biol; citation_title=Robust and sensitive control of a quorum-sensing circuit by two interlocked feedback loops; citation_author=JW Williams; citation_author=X Cui; citation_author=A Levchenko; citation_author=AM Stevens; citation_volume=4; citation_issue=234; citation_publication_date=2008; citation_pages=234; citation_id=CR10;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Appl Environ Microbiol; citation_title=Implications of rewiring bacterial quorum sensing; citation_author=EL Haseltine; citation_author=FH Arnold; citation_volume=74; citation_issue=2; citation_publication_date=2008; citation_pages=437-45; citation_doi=10.1128/AEM.01688-07; citation_id=CR11;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Microbiol; citation_title=Heterogeneity in quorum sensing-regulated bioluminescence of Vibrio harveyi; citation_author=C Anetzberger; citation_author=T Pirch; citation_author=K Jung; citation_volume=73; citation_issue=2; citation_publication_date=2009; citation_pages=267-77; citation_doi=10.1111/j.1365-2958.2009.06768.x; citation_id=CR12;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS One; citation_title=Heterogeneous response to a quorum-sensing signal in the luminescence of individual Vibrio fischeri; citation_author=PD P&#233;rez; citation_author=SJ Hagen; citation_volume=5; citation_issue=11; citation_publication_date=2010; citation_pages=e15473; citation_doi=10.1371/journal.pone.0015473; citation_id=CR13;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Angew Chem Int Ed Engl; citation_title=Microfluidic confinement of single cells of bacteria in small volumes initiates high-density behavior of quorum sensing and growth and reveals its variability; citation_author=JQ Boedicker; citation_author=ME Vincent; citation_author=RF Ismagilov; citation_volume=48; citation_issue=32; citation_publication_date=2009; citation_pages=5908-11; citation_doi=10.1002/anie.200901550; citation_id=CR14;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Biol Phys; citation_title=Bacterium in a box: sensing of quorum and environment by the LuxI/LuxR gene regulatory circuit; citation_author=SJ Hagen; citation_author=M Son; citation_author=JT Weiss; citation_author=JH Young; citation_volume=36; citation_issue=3; citation_publication_date=2010; citation_pages=317-327; citation_doi=10.1007/s10867-010-9186-4; citation_id=CR15;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Stochastic models for regulatory networks of the genetic toggle switch; citation_author=T Tian; citation_author=K Burrage; citation_volume=103; citation_issue=22; citation_publication_date=2006; citation_pages=8372-7; citation_doi=10.1073/pnas.0507818103; citation_id=CR16;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BMC Syst Biol; citation_title=Noise-induced switches in network systems of the genetic toggle switch; citation_author=J Wang; citation_author=J Zhang; citation_author=Z Yuan; citation_author=T Zhou; citation_volume=1; citation_publication_date=2007; citation_pages=50; citation_doi=10.1186/1752-0509-1-50; citation_id=CR17;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS One; citation_title=Asymmetric stochastic switching driven by intrinsic molecular noise; citation_author=D Frigola; citation_author=L Casanellas; citation_author=JM Sancho; citation_author=M Iba&#241;es; citation_volume=7; citation_issue=2; citation_publication_date=2012; citation_pages=e31407; citation_doi=10.1371/journal.pone.0031407; citation_id=CR18;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nature; citation_title=A synchronized quorum of genetic clocks; citation_author=T Danino; citation_author=O Mondrag&#243;n-Palomino; citation_author=L Tsimring; citation_author=J Hasty; citation_volume=463; citation_issue=7279; citation_publication_date=2010; citation_pages=326-330; citation_doi=10.1038/nature08753; citation_id=CR19;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing; citation_author=J Garc&#237;a-Ojalvo; citation_author=M Elowitz; citation_author=S Strogatz; citation_volume=101; citation_issue=30; citation_publication_date=2004; citation_pages=10955; citation_doi=10.1073/pnas.0307095101; citation_id=CR20;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BioSystems; citation_title=Modeling the regulation of the competence-evoking quorum sensing network in Streptococcus pneumoniae; citation_author=D Karlsson; citation_author=S Karlsson; citation_author=E Gustafsson; citation_author=BH Normark; citation_author=P Nilsson; citation_volume=90; citation_publication_date=2007; citation_pages=211-23; citation_doi=10.1016/j.biosystems.2006.08.005; citation_id=CR21;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Theor Biol; citation_title=Interplay of two quorum sensing regulation systems of Vibrio fischeri; citation_author=C Kuttler; citation_author=BA Hense; citation_volume=251; citation_publication_date=2008; citation_pages=167-80; citation_doi=10.1016/j.jtbi.2007.11.015; citation_id=CR22;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS Comput Biol; citation_title=Noise reduction by diffusional dissipation in a minimal quorum sensing motif; citation_author=Y Tanouchi; citation_author=D Tu; citation_author=J Kim; citation_author=L You; citation_volume=4; citation_issue=8; citation_publication_date=2008; citation_pages=e1000167; citation_doi=10.1371/journal.pcbi.1000167; citation_id=CR23;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Syst Biol; citation_title=Optimal tuning of bacterial sensing potential; citation_author=A Pai; citation_author=L You; citation_volume=5; citation_issue=286; citation_publication_date=2009; citation_pages=286; citation_id=CR24;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS Comput Biol; citation_title=Transition to quorum sensing in an Agrobacterium population: A stochastic model; citation_author=AB Goryachev; citation_author=DJ Toh; citation_author=KB Wee; citation_author=HB Zhang; citation_author=LH Zhang; citation_author=T Lee; citation_volume=1; citation_issue=4; citation_publication_date=2005; citation_pages=e37; citation_doi=10.1371/journal.pcbi.0010037; citation_id=CR25;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Artif Life; citation_title=A model of the quorum sensing system in Vibrio fischeri using P systems; citation_author=FJ Romero-Campero; citation_author=MJ P&#233;rez-Jim&#233;nez; citation_volume=14; citation_publication_date=2008; citation_pages=95-109; citation_doi=10.1162/artl.2008.14.1.95; citation_id=CR26;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS One; citation_title=Timing cellular decision making under noise via cell-cell communication; citation_author=A Koseska; citation_author=A Zaikin; citation_author=J Kurths; citation_author=J Garc&#237;a-Ojalvo; citation_volume=4; citation_issue=3; citation_publication_date=2009; citation_pages=e4872; citation_doi=10.1371/journal.pone.0004872; citation_id=CR27;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS Comput Biol; citation_title=A cell-based model for quorum sensing in heterogeneous bacterial colonies; citation_author=P Melke; citation_author=P Sahlin; citation_author=A Levchenko; citation_author=H J&#246;nsson; citation_volume=6; citation_issue=6; citation_publication_date=2010; citation_pages=e1000819; citation_doi=10.1371/journal.pcbi.1000819; citation_id=CR28;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Bacteriol; citation_title=Bright mutants of Vibrio fischeri ES114 reveal conditions and regulators that control bioluminescence and expression of the lux operon; citation_author=NL Lyell; citation_author=AK Dunn; citation_author=JL Bose; citation_author=EV Stabb; citation_volume=192; citation_issue=19; citation_publication_date=2010; citation_pages=5103-14; citation_doi=10.1128/JB.00524-10; citation_id=CR29;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol Syst Biol; citation_title=Autoinducer-independent mutants of the LuxR transcriptional activator exhibit differential effects on the twolux promoters ofVibrio fischeri; citation_author=D Sitnikov; citation_author=G Shadel; citation_author=T Baldwin; citation_volume=252; citation_issue=5; citation_publication_date=1996; citation_pages=622-625; citation_id=CR30;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Biol Chem; citation_title=Identification of a distantly located regulatory element in the luxD gene required for negative autoregulation of the Vibrio fischeri luxR gene; citation_author=G Shadel; citation_author=T Baldwin; citation_volume=267; citation_issue=11; citation_publication_date=1992; citation_pages=7690; citation_id=CR31;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Phys Chem; citation_title=Exact stochastic simulation of coupled chemical reactions; citation_author=D Gillespie; citation_volume=81; citation_issue=25; citation_publication_date=1977; citation_pages=2340-2361; citation_doi=10.1021/j100540a008; citation_id=CR32;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Sci (New York, N.Y.); citation_title=Gene regulation at the single-cell level; citation_author=N Rosenfeld; citation_author=JW Young; citation_author=U Alon; citation_author=PS Swain; citation_author=MB Elowitz; citation_volume=307; citation_issue=5717; citation_publication_date=2005; citation_pages=1962-5; citation_doi=10.1126/science.1106914; citation_id=CR33;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Biophys J; citation_title=Interplay between intrinsic noise and the stochasticity of the cell cycle in bacterial colonies; citation_author=O Canela-Xandri; citation_author=F Sagu&#233;s; citation_author=J Buceta; citation_volume=98; citation_issue=11; citation_publication_date=2010; citation_pages=2459-68; citation_doi=10.1016/j.bpj.2010.02.045; citation_id=CR34;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Syst Biol (Stevenage); citation_title=Cellular growth and division in the Gillespie algorithm; citation_author=T Lu; citation_author=D Volfson; citation_author=L Tsimring; citation_author=J Hasty; citation_volume=1; citation_publication_date=2004; citation_pages=121; citation_doi=10.1049/sb:20045016; citation_id=CR35;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nat Rev Genet; citation_title=Stochasticity in gene expression: from theories to phenotypes; citation_author=M K&#230;rn; citation_author=T Elston; citation_author=W Blake; citation_author=J Collins; citation_volume=6; citation_issue=6; citation_publication_date=2005; citation_pages=451-464; citation_doi=10.1038/nrg1615; citation_id=CR36;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Sci (New York, N.Y.); citation_title=Probing gene expression in live cells, one protein molecule at a time; citation_author=J Yu; citation_author=J Xiao; citation_author=X Ren; citation_author=K Lao; citation_author=XS Xie; citation_volume=311; citation_issue=5767; citation_publication_date=2006; citation_pages=1600-3; citation_doi=10.1126/science.1119623; citation_id=CR37;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nature; citation_title=Stochastic protein expression in individual cells at the single molecule level; citation_author=L Cai; citation_author=N Friedman; citation_author=XS Xie; citation_volume=440; citation_issue=7082; citation_publication_date=2006; citation_pages=358-62; citation_doi=10.1038/nature04599; citation_id=CR38;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nat Genet; citation_title=Regulation of noise in the expression of a single gene; citation_author=EM Ozbudak; citation_author=M Thattai; citation_author=I Kurtser; citation_author=AD Grossman; citation_author=A van Oudenaarden; citation_volume=31; citation_publication_date=2002; citation_pages=69-73; citation_doi=10.1038/ng869; citation_id=CR39;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nat Rev Genet; citation_title=Microfluidic devices for measuring gene network dynamics in single cells; citation_author=MR Bennett; citation_author=J Hasty; citation_volume=10; citation_issue=9; citation_publication_date=2009; citation_pages=628-38; citation_doi=10.1038/nrg2625; citation_id=CR40;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Bacteriol; citation_title=Reversible acyl-homoserine lactone binding to purified Vibrio fischeri LuxR protein; citation_author=M Urbanowski; citation_author=C Lostroh; citation_author=E Greenberg; citation_volume=186; citation_issue=3; citation_publication_date=2004; citation_pages=631; citation_doi=10.1128/JB.186.3.631-637.2004; citation_id=CR41;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Proc Natl Acad Sci U S A; citation_title=Revisiting quorum sensing: discovery of additional chemical and biological functions for 3-oxo-N-acylhomoserine lactones; citation_author=G Kaufmann; citation_author=R Sartorio; citation_author=S Lee; citation_author=C Rogers; citation_author=M Meijler; citation_author=J Moss; citation_author=B Clapham; citation_author=A Brogan; citation_author=T Dickerson; citation_author=K Janda; citation_volume=102; citation_issue=2; citation_publication_date=2005; citation_pages=309; citation_doi=10.1073/pnas.0408639102; citation_id=CR42;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Bacteriol; citation_title=Characterizing the effect of the Staphylococcus aureus virulence factor regulator, SarA, on log-phase mRNA half-lives; citation_author=C Roberts; citation_author=KL Anderson; citation_author=E Murphy; citation_author=SJ Projan; citation_author=W Mounts; citation_author=B Hurlburt; citation_author=M Smeltzer; citation_author=R Overbeek; citation_author=T Disz; citation_author=PM Dunman; citation_volume=188; citation_issue=7; citation_publication_date=2006; citation_pages=2593-603; citation_doi=10.1128/JB.188.7.2593-2603.2006; citation_id=CR43;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J Bacteriol; citation_title=Diffusion of autoinducer is involved in regulation of the Vibrio fischeri luminescence system; citation_author=H Kaplan; citation_author=E Greenberg; citation_volume=163; citation_issue=3; citation_publication_date=1985; citation_pages=1210; citation_id=CR44;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Phys Biol; citation_title=Timing the start of division in E. coli: a single-cell study; citation_author=G Reshes; citation_author=S Vanounou; citation_author=I Fishov; citation_author=M Feingold; citation_volume=5; citation_issue=4; citation_publication_date=2008; citation_pages=046001; citation_doi=10.1088/1478-3975/5/4/046001; citation_id=CR45;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Arch Microbiol; citation_title=Exponential growth of Escherichia coli B/r during its division cycle is demonstrated by the size distribution in liquid culture; citation_author=F Trueba; citation_author=L Koppes; citation_volume=169; citation_publication_date=1998; citation_pages=491-496; citation_doi=10.1007/s002030050601; citation_id=CR46;'/>
    
    <meta name="citation_reference" content='citation_journal_title=J theor biol; citation_title=Extracellular noise-induced stochastic synchronization in heterogeneous quorum sensing network; citation_author=D Hong; citation_author=WM Saidel; citation_author=S Man; citation_author=JV Martin; citation_volume=245; citation_issue=4; citation_publication_date=2007; citation_pages=726-736; citation_doi=10.1016/j.jtbi.2006.12.006; citation_id=CR47;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Phys rev lett; citation_title=Molecular communication through stochastic synchronization induced by extracellular fluctuations; citation_author=T Zhou; citation_author=L Chen; citation_author=K Aihara; citation_volume=95; citation_issue=17; citation_publication_date=2005; citation_pages=178103; citation_doi=10.1103/PhysRevLett.95.178103; citation_id=CR48;'/>
    
    <meta name="citation_reference" content='citation_title=Noise-induced transitions: theory and applications in physics, chemistry and biology; citation_author=W Horsthemke; citation_author=R Lefever; citation_publisher=Springer-Verlag; citation_publication_date=1984; citation_id=CR49;'/>
    
    <meta name="citation_reference" content='citation_journal_title=BMC Syst Biol; citation_title=Noise and crosstalk in two quorum-sensing inputs of Vibrio fischeri; citation_author=PD P&#233;rez; citation_author=JT Weiss; citation_author=SJ Hagen; citation_volume=5; citation_publication_date=2011; citation_pages=153; citation_doi=10.1186/1752-0509-5-153; citation_id=CR50;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Biophys J; citation_title=Measurement of the copy number of the master quorum-sensing regulator of a bacterial cell; citation_author=SW Teng; citation_author=Y Wang; citation_author=KC Tu; citation_author=T Long; citation_author=P Mehta; citation_author=NS Wingreen; citation_author=BL Bassler; citation_author=NP Ong; citation_volume=98; citation_issue=9; citation_publication_date=2010; citation_pages=2024-31; citation_doi=10.1016/j.bpj.2010.01.031; citation_id=CR51;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Mol cell; citation_title=Negative feedback loops involving small regulatory RNAs precisely control the Vibrio harveyi quorum-sensing response; citation_author=KC Tu; citation_author=T Long; citation_author=SL Svenningsen; citation_author=NS Wingreen; citation_author=BL Bassler; citation_volume=37; citation_issue=4; citation_publication_date=2010; citation_pages=567-79; citation_doi=10.1016/j.molcel.2010.01.022; citation_id=CR52;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Cell; citation_title=Cellular decision making and biological noise: from microbes to mammals; citation_author=G Bal&#225;zsi; citation_author=A van Oudenaarden; citation_author=JJ Collins; citation_volume=144; citation_issue=6; citation_publication_date=2011; citation_pages=910-25; citation_doi=10.1016/j.cell.2011.01.030; citation_id=CR53;'/>
    
    <meta name="citation_reference" content='citation_journal_title=Nature; citation_title=Enhancement of cellular memory by reducing stochastic transitions; citation_author=M Acar; citation_author=A Becskei; citation_author=A van Oudenaarden; citation_volume=435; citation_issue=7039; citation_publication_date=2005; citation_pages=228-32; citation_doi=10.1038/nature03524; citation_id=CR54;'/>
    
    <meta name="citation_reference" content='citation_journal_title=PLoS One; citation_title=Speed-dependent cellular decision making in nonequilibrium genetic circuits; citation_author=NR Nen&#233;; citation_author=J Garc&#237;a-Ojalvo; citation_author=A Zaikin; citation_volume=7; citation_issue=3; citation_publication_date=2012; citation_pages=e32779; citation_doi=10.1371/journal.pone.0032779; citation_id=CR55;'/>
    

    <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge"/>
<meta name="viewport" content="width=device-width, initial-scale=1"/>

    <link rel="apple-touch-icon" sizes="57x57"
          href="/static/images/apple-touch-icon-57x57.511c7661.png"/>
    <link rel="apple-touch-icon" sizes="60x60"
          href="/static/images/apple-touch-icon-60x60.d80aabfd.png"/>
    <link rel="apple-touch-icon" sizes="72x72"
          href="/static/images/apple-touch-icon-72x72.2f8e1a72.png"/>
    <link rel="apple-touch-icon" sizes="76x76"
          href="/static/images/apple-touch-icon-76x76.d33ae418.png"/>
    <link rel="apple-touch-icon" sizes="114x114"
          href="/static/images/apple-touch-icon-114x114.c149133d.png"/>
    <link rel="apple-touch-icon" sizes="120x120"
          href="/static/images/apple-touch-icon-120x120.16840646.png"/>
    <link rel="apple-touch-icon" sizes="144x144"
          href="/static/images/apple-touch-icon-144x144.4f2b1ad4.png"/>
    <link rel="apple-touch-icon" sizes="152x152"
          href="/static/images/apple-touch-icon-152x152.344ef152.png"/>
    <link rel="apple-touch-icon" sizes="180x180"
          href="/static/images/apple-touch-icon-180x180.324ae1a6.png"/>
    <link rel="icon" type="image/png"
          href="/static/images/android-chrome-192x192.219f289c.png" sizes="192x192"/>
    <link rel="icon" type="image/png" href="/static/images/favicon-16x16.dc83f443.png"
          sizes="16x16"/>
    <link rel="icon" type="image/png" href="/static/images/favicon-32x32.44498c19.png"
          sizes="32x32"/>
    <link rel="icon" type="image/png" href="/static/images/favicon-96x96.62c11b8a.png"
          sizes="96x96"/>
    <link rel="icon" type="image/png" href="/static/images/favicon-194x194.578b1b58.png"
          sizes="194x194"/>
    <link rel="manifest" href="/static/images/favicons/biomedcentral/manifest.json"/>
    <link rel="shortcut icon" href="/static/images/favicon.dc10a51d.ico"/>
    <meta name="msapplication-TileColor" content="#da532c"/>
    <meta name="msapplication-config"
          content="/static/images/favicons/biomedcentral/browserconfig.xml"/>
    <meta name="msapplication-TileImage"
          content="/static/images/mstile-144x144.9d93277a.png"/>
    <meta name="theme-color" content="#ffffff"/>


<script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
<script>(function(d) {if (sessionStorage.getItem('webfonts') == 'true'){d.className += ' webfonts-loaded';}})(document.documentElement);</script>

    <link rel="stylesheet" media="screen" href="/static/css/biomedcentral.bbacbecb.css"/>
    <link rel="stylesheet" media="print" href="/static/css/biomedcentral-print.12735a02.css"/>


<script>var _gaq = [];</script>

    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcsystbiol.biomedcentral.com',
            siteWithPath: 'bmcsystbiol.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcsystbiol',
            cmsPrefix: 'http://studio-cms.springer.com/studio/',
            
            doi: '10.1186/1752-0509-7-6',
            
            publisherBrand: 'BioMed Central'
        };
    </script>


    <script>
        (function (i, s, o, g, r, a, m) {
            i['GoogleAnalyticsObject'] = r;
            i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
            a = s.createElement(o),
                    m = s.getElementsByTagName(o)[0];
            a.async = 1;
            a.src = g;
            m.parentNode.insertBefore(a, m)
        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

        var sampleRate = 20;

            
            ga('create', "UA-62065396-20",
                    {
                        cookieDomain: 'none',
                        allowLinker: true,
                        siteSpeedSampleRate: sampleRate
                    });
            ga("set", "anonymizeIp", true);
            ga('send', 'pageview');
            

        ga('create', "UA-54492316-9",
                {
                    cookieDomain: 'none',
                    allowLinker: true,
                    siteSpeedSampleRate: sampleRate,
                    name: 'global'
                });

        ga("global.set", "anonymizeIp", true);
        ga('global.send', 'pageview');
    </script>



    
    <link rel="canonical" href="//bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-6"/>
    


    
    <meta property="og:url" content="http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-7-6"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="BMC Systems Biology"/>
    <meta property="og:title" content="Dynamics of the quorum sensing switch: stochastic and non-stationary effects"/>
    <meta property="og:description" content="A wide range of bacteria species are known to communicate through the so called quorum sensing (QS) mechanism by means of which they produce a small molecule that can freely diffuse in the environment and in the cells. Upon reaching a threshold concentration, the signalling molecule activates the QS-controlled genes that promote phenotypic changes. This mechanism, for its simplicity, has become the model system for studying the emergence of a global response in prokaryotic cells. Yet, how cells precisely measure the signal concentration and act coordinately, despite the presence of fluctuations that unavoidably affects cell regulation and signalling, remains unclear. We propose a model for the QS signalling mechanism in Vibrio fischeri based on the synthetic strains lux01 and lux02. Our approach takes into account the key regulatory interactions between LuxR and LuxI, the autoinducer transport, the cellular growth and the division dynamics. By using both deterministic and stochastic models, we analyze the response and dynamics at the single-cell level and compare them to the global response at the population level. Our results show how fluctuations interfere with the synchronization of the cell activation and lead to a bimodal phenotypic distribution. In this context, we introduce the concept of precision in order to characterize the reliability of the QS communication process in the colony. We show that increasing the noise in the expression of LuxR helps cells to get activated at lower autoinducer concentrations but, at the same time, slows down the global response. The precision of the QS switch under non-stationary conditions decreases with noise, while at steady-state it is independent of the noise value. Our in silico experiments show that the response of the LuxR/LuxI system depends on the interplay between non-stationary and stochastic effects and that the burst size of the transcription/translation noise at the level of LuxR controls the phenotypic variability of the population. These results, together with recent experimental evidences on LuxR regulation in wild-type species, suggest that bacteria have evolved mechanisms to regulate the intensity of those fluctuations."/>
    <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12918.jpg"/>

</head>
<body class="fulltext">
    <div class="container" style="height: 0"><div id="wt_refpoint"></div></div>
    <p class="skip-to-main">
    <a class="skip-to-main__link button button--priority-2 button--size-200" href="#main">Skip to main content</a>
</p>
    <div id="top">
        
<div id="RealMediaLeaderboard" class="adsbox RealMediaAd Ad Ad--leaderboard" data-position="Top">
    <div class="Ad_content">
        <p class="Ad_label">Advertisement</p>
        
    </div>
</div>


         
            <header class="Header" role="banner">
                <div class="Header_inner">
                    <a id="header-logo" href="//www.biomedcentral.com/" class="logo" title="BioMed Central">
                        <img srcset="/static/images/biomed-central-logo_180x35.85951291.svg" src="/static/images/biomed-central-logo_180x35.1c7100cf.png" class="logo__img" alt="Biomed Central" />
                    </a>
                    <div class="Header_right">
                        <div class="Toggles">
                            <span class="Toggles_item NavToggle" data-toggle="menu">Menu</span>
                            <span class="Toggles_item SearchToggle" data-toggle="search">
                                <span class="SearchToggle_icon" aria-hidden="true"></span>
                            </span>
                        </div>
                        <form role="search" id="globalSearch" class="Search Search--expandable" method="GET" action="//www.biomedcentral.com/search" data-track-event="submit"
      data-event-category="Search and Results" data-event-action="Submit search" data-event-label="">
    <div class="Search_content">
        <label for="publisherSearch" class="Search_label ">Search</label>
        <input id="publisherSearch" class="Search_input" autocomplete="off" role="textbox" name="query" type="text" value="" title="Search BioMed Central"/>
        <button id="search" class="Search_btn" type="submit">
            <span class="u-srOnly">Search</span>
            <span class="Search_icon"></span>
        </button>
        <input type=hidden name="searchType" value="publisherSearch"/>
    </div>
</form>

                    </div>
                    
    <nav id="globalNavigation" class="GlobalNav js-fadeToggle" role="navigation" aria-label="publisher">
        <h2 class="u-srOnly">Publisher main menu</h2>
        <ul id="publisherNavigation" class="GlobalNav_list">
            
            <li>
                <a class="" href="//www.biomedcentral.com/journals">Explore journals</a>
            </li>
            
            <li>
                <a class="" href="//www.biomedcentral.com/getpublished">Get published</a>
            </li>
            
            <li>
                <a class="" href="//www.biomedcentral.com/about">About BioMed Central</a>
            </li>
            
        </ul>
        <p id="loginLink" class="GlobalNav_sign-in sign-in" data-replace="true" data-placeholder="/placeholder/v1/username">
            <a class="sign-in__link" href="//www.biomedcentral.com/my-account" data-track-event="click" data-event-category="Account" data-event-action="Click My Account">
                <span class="sign-in__no-js-account-hide">Login<span class="sign-in__js-account-hide"> to </span></span><span class="sign-in__js-account-hide sign-in__no-js-capitalise">your account</span>
            </a>
        </p>
    </nav>

                </div>
            </header>
            <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
        

        <main role="main">
            
            
            <header class="journal-header" role="banner">
                <div class="journal-header__inner">
                    <h2 class="journal-header__title journal-header__title--medium" id="journalTitle">
                        <a class="journal-header__link" href="/">BMC Systems Biology</a>
                    </h2>
                </div>
                <div class="navbar">
                    <div class="navbar__inner">
                        

                        <nav class="navbar__nav">
                            <h3 class="navbar__title" data-toggle="collapse" data-target="journal-menu">
                                <span class="sr-only">Main </span>menu
                            </h3>
                            <ul id="journal-menu" class="navbar__menu" id="siteNavigation" role="menu">
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/'>Home</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/about'>About</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link is-active" href='/articles'>Articles</a>
                                    </li>
                                
                                    <li class="navbar__item" role="menuitem">
                                        <a class="navbar__link" href='/submission-guidelines'>Submission Guidelines</a>
                                    </li>
                                
                            </ul>
                        </nav>
                    </div>
                </div>
                <div class="journal-identity journal-identity--networks-aqua">
                    <div class="journal-identity__container"></div>
                </div>
            </header>
            
        
            <div class="container">
                
                <div id="main" class="grid">
                
                    <div class="main__content grid__col grid__col--xs-12 grid__col--md-9" id="main-content">
                        <div class="block" id="Test-ImgSrc">
                            
                            
                            
    <div class="FulltextWrapper">
        <div class="Categories">
            <div class="ArticleCategory">Research article</div>
            <div class="OpenAccessLabel">Open Access</div>
            
        </div>
        
    </div>

                            
                            <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" lang="en">Dynamics of the quorum sensing switch: stochastic and non-stationary effects</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames"><ul class="u-listReset"><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">Marc&nbsp;Weber</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author" data-jumpto="Aff"><span xmlns="http://www.w3.org/1999/xhtml" class="AuthorName">Javier&nbsp;Buceta</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:javier.buceta@pcb.ub.es" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Systems Biology</span><span class="ArticleCitation_Year">2013</span><span class="ArticleCitation_Volume"><strong>7</strong>:6</span></span></div><p class="ArticleDOI"><strong>DOI: </strong>10.1186/1752-0509-7-6</p><p class="Copyright">Â© &nbsp;Weber and Buceta; licensee BioMed Central Ltd.&nbsp;2013</p></div></div><div class="ArticleHistory"><p class="History HistoryReceived"><strong>Received: </strong>4&nbsp;July&nbsp;2012</p><p class="History HistoryAccepted"><strong>Accepted: </strong>7&nbsp;January&nbsp;2013</p><p class="History HistoryOnlineDate"><strong>Published: </strong>16&nbsp;January&nbsp;2013</p></div></div>
                            
                            
                            
    

                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract" id="Abs1" lang="en"><h2 class="Heading js-ToggleCollapseSection">Abstract</h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">A wide range of bacteria species are known to communicate through the so called quorum sensing (QS) mechanism by means of which they produce a small molecule that can freely diffuse in the environment and in the cells. Upon reaching a threshold concentration, the signalling molecule activates the QS-controlled genes that promote phenotypic changes. This mechanism, for its simplicity, has become the model system for studying the emergence of a global response in prokaryotic cells. Yet, how cells precisely measure the signal concentration and act coordinately, despite the presence of fluctuations that unavoidably affects cell regulation and signalling, remains unclear.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We propose a model for the QS signalling mechanism in <em xmlns="" class="EmphasisTypeItalic">Vibrio fischeri</em> based on the synthetic strains <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em>. Our approach takes into account the key regulatory interactions between LuxR and LuxI, the autoinducer transport, the cellular growth and the division dynamics. By using both deterministic and stochastic models, we analyze the response and dynamics at the single-cell level and compare them to the global response at the population level. Our results show how fluctuations interfere with the synchronization of the cell activation and lead to a bimodal phenotypic distribution. In this context, we introduce the concept of precision in order to characterize the reliability of the QS communication process in the colony. We show that increasing the noise in the expression of LuxR helps cells to get activated at lower autoinducer concentrations but, at the same time, slows down the global response. The precision of the QS switch under non-stationary conditions decreases with noise, while at steady-state it is independent of the noise value.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">Our <em xmlns="" class="EmphasisTypeItalic">in silico</em> experiments show that the response of the LuxR/LuxI system depends on the interplay between non-stationary and stochastic effects and that the burst size of the transcription/translation noise at the level of LuxR controls the phenotypic variability of the population. These results, together with recent experimental evidences on LuxR regulation in wild-type species, suggest that bacteria have evolved mechanisms to regulate the intensity of those fluctuations.</p>
              </div><div xmlns="http://www.w3.org/1999/xhtml" class="KeywordGroup" lang="en">
              <h3 xmlns="" class="Heading">Keywords</h3>
              <span class="Keyword">Quorum sensing</span>
              <span class="Keyword">Noise</span>
              <span class="Keyword">Stochastic modeling</span>
              <span class="Keyword">Vibrio fischeri</span>
              <span class="Keyword">Autoinducer</span>
              <span class="Keyword">Synchronization</span>
            </div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Background</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Bacteria, long thought having a solitary existence, were found to communicate with one another by sending and receiving chemical messages [<span class="CitationRef"><a href="#CR1">1</a></span>]. Their communication mechanism results in the ability to synchronize the activity of the colony as a whole. The latter leads to a coordinated behaviour that in some cases resembles that of multicellular organisms, e.g. the so-called community effect during development [<span class="CitationRef"><a href="#CR2">2</a></span>]. Thus, by means of the quorum sensing (QS) mechanism, cells produce, export, and import signalling molecules (autoinducer). As the colony grows, more cells produce and export autoinducer, leading to an increasing concentration of the signalling molecule in the environment and in the cells. Upon reaching a concentration threshold, the autoinducer activates the expression of QS-controlled genes therefore coordinating the cells in a density-dependent manner. Importantly, QS controls a number of relevant phenotypic changes in bacteria as for example the virulence in <em xmlns="" class="EmphasisTypeItalic">S. aureus</em>[<span class="CitationRef"><a href="#CR3">3</a></span>]. In addition, it has become a model system for studying the emergence of coordinated behaviour in communicating cells. All in all, QS has opened a research field with promising technological applications [<span class="CitationRef"><a href="#CR4">4</a></span>], as for example, the environmentally controlled invasion of cancer cells [<span class="CitationRef"><a href="#CR5">5</a></span>].</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">The QS systems in gram-negative bacteria share a core network architecture. In this regard, a characteristic model system is the LuxR/LuxI regulatory network in <em xmlns="" class="EmphasisTypeItalic">Vibrio fischeri</em>[<span class="CitationRef"><a href="#CR6">6</a></span>]. LuxR protein is an autoinducer-dependent activator of the lux operon that drives the autocatalytic expression of <em xmlns="" class="EmphasisTypeItalic">luxR</em> and of the autoinducer synthase, <em xmlns="" class="EmphasisTypeItalic">luxI</em>, together with that of the genes responsible for the production of bioluminescence. The up-regulation of <em xmlns="" class="EmphasisTypeItalic">luxI</em> increases the production of autoinducer molecules that in turn activates further gene expression. The resulting positive feedback loop leads to a bistable switch-like behaviour depending on the concentration of the autoinducer as shown by <em xmlns="" class="EmphasisTypeItalic">in silico</em>[<span class="CitationRef"><a href="#CR7">7</a></span>â<span class="CitationRef"><a href="#CR9">9</a></span>] and <em xmlns="" class="EmphasisTypeItalic">in vivo</em> experiments [<span class="CitationRef"><a href="#CR10">10</a></span>, <span class="CitationRef"><a href="#CR11">11</a></span>]. Such switch-like behaviour has been observed at the population level by measuring the average gene expression level. However, how individual cells behave remains puzzling. In fact, as observed in <em xmlns="" class="EmphasisTypeItalic">Vibrio harveyi</em>[<span class="CitationRef"><a href="#CR12">12</a></span>], <em xmlns="" class="EmphasisTypeItalic">Vibrio fischeri</em>[<span class="CitationRef"><a href="#CR13">13</a></span>], <em xmlns="" class="EmphasisTypeItalic">Pseudomonas aeruginosa</em>[<span class="CitationRef"><a href="#CR14">14</a></span>], and <em xmlns="" class="EmphasisTypeItalic">luxI</em>/<em xmlns="" class="EmphasisTypeItalic">luxR-GFP</em> strains of <em xmlns="" class="EmphasisTypeItalic">E. coli</em>[<span class="CitationRef"><a href="#CR15">15</a></span>], the cellular response to QS signals seems to be highly heterogeneous at the level of the distribution of both the population phenotype and the response times of individual cells.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">A number of studies have shown that noise plays an important role in bistable systems [<span class="CitationRef"><a href="#CR16">16</a></span>â<span class="CitationRef"><a href="#CR18">18</a></span>]. Therefore, the aforementioned heterogeneity may be caused by the random fluctuations that unavoidably affect cell regulation and signalling. This poses the intriguing question of how cells achieve a coordinated response in the presence of noise. Indeed, the QS mechanism may produce a robust and synchronized behaviour at the level of the population both experimentally [<span class="CitationRef"><a href="#CR19">19</a></span>] and theoretically [<span class="CitationRef"><a href="#CR20">20</a></span>]. However, how this behaviour at the collective level arises from the stochastic dynamics of individual cells is still an open question. At the end, in the framework of QS, a collective response means a precise information exchange in the colony. Consequently, how can a bacterial population estimate its number of constituents precisely if such information is fuzzy at the single cell level? Herein, we shed light on this problem and investigate how noise affects the QS transition both at the level of individual cells and at the level of the cell population.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">In the context of QS modelling, most research has focused on the understanding of the intracellular circuit [<span class="CitationRef"><a href="#CR7">7</a></span>â<span class="CitationRef"><a href="#CR11">11</a></span>, <span class="CitationRef"><a href="#CR21">21</a></span>â<span class="CitationRef"><a href="#CR24">24</a></span>], i.e. single cell studies, while few of them have considered an ensemble of communicating cells [<span class="CitationRef"><a href="#CR25">25</a></span>â<span class="CitationRef"><a href="#CR28">28</a></span>]. Yet, so far no study has taken into account the coupling of the signalling mechanism at the single cell and collective levels by stochastic means together with realistic dynamics of the proliferation process. In this work, we model the QS mechanism by using both deterministic and stochastic approaches and taking into account the key regulatory interactions between LuxR and LuxI, the autoinducer transport, the cellular growth and the division dynamics. Our results indicate that the cell response is highly heterogeneous and that noise in the gene expression of <em xmlns="" class="EmphasisTypeItalic">luxR</em> is the main factor that determines this variability. Moreover, we show that the transition of the QS switch near the critical concentration of autoinducer is very slow compared to other characteristic temporal scales of the process and that, as a consequence, the non-stationary effects are crucial for setting a precise switch. As we show further below, the dilution due to cell growth and division is a key element required for an in-depth understanding of the QS response dynamics. In addition, we demonstrate that noise, depending on the cell density, can either prevent or promote phenotypic changes indicating a beneficial role played by stochasticity. Altogether, we find that the precision of the QS switch for determining the number of cells in the colony is highly dynamic and context dependent, which in turn favors adaptability.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Methods</h2><div class="js-CollapseSection"><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">Modelling of the LuxI/LuxR gene regulatory network</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The regulatory interactions that control the wild-type lux operon are more complex than first thought [<span class="CitationRef"><a href="#CR29">29</a></span>]. Those include both positive and negative regulation of the <em xmlns="" class="EmphasisTypeItalic">luxR</em> gene depending on the concentration of the autoinducer [<span class="CitationRef"><a href="#CR30">30</a></span>]. Simplified synthetic constructs, such as <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em>[<span class="CitationRef"><a href="#CR10">10</a></span>], retain the minimal <em xmlns="" class="EmphasisTypeItalic">luxI/luxR</em> regulatory motif and lack the structural genes responsible for light emission that may also play a regulatory role, e.g. <em xmlns="" class="EmphasisTypeItalic">luxD</em>[<span class="CitationRef"><a href="#CR31">31</a></span>]. Still, these constructs reproduce the main features of the wild-type operon as revealed by GFP tags reporting the promoter activity [<span class="CitationRef"><a href="#CR10">10</a></span>]. In addition, <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> constructs allow to perform controlled experiments that have shed light on the wild-type dynamics and its regulatory interactions. Herein, we follow this approach and focus on the <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> constructs as well characterized examples of the behaviour of the wild-type operon. The <em xmlns="" class="EmphasisTypeItalic">lux01</em> operon lacks the <em xmlns="" class="EmphasisTypeItalic">luxI</em> gene and only <em xmlns="" class="EmphasisTypeItalic">gfp</em> is transcribed in that direction. On the other hand, the <em xmlns="" class="EmphasisTypeItalic">lux02</em> operon carries a <em xmlns="" class="EmphasisTypeItalic">luxI::gfp</em> fusion. Accordingly, <em xmlns="" class="EmphasisTypeItalic">lux01</em> cells cannot produce their own autoinducer and the induction in that case is driven by adding exogenous autoinducer to the medium. Figure <span class="InternalRef"><a href="#Fig1">1</a></span> shows schematically the regulatory interactions we consider in our model. The autoinducer molecules (<em xmlns="" class="EmphasisTypeItalic">A</em>) are produced due to the action of their synthetase, LuxI, and bind to the cytoplasmic protein LuxR (<em xmlns="" class="EmphasisTypeItalic">R</em>) creating a complex (<em xmlns="" class="EmphasisTypeItalic">C</em><sub>2</sub>). The latter binds to the promoter region activating both the transcription of <em xmlns="" class="EmphasisTypeItalic">luxI::gfp</em> (only <em xmlns="" class="EmphasisTypeItalic">gfp</em> in the case of <em xmlns="" class="EmphasisTypeItalic">lux01</em>) and <em xmlns="" class="EmphasisTypeItalic">luxR</em>. Signalling molecules can diffuse passively in and out the cell and contribute to increase the external concentration of the autoinducer (<em xmlns="" class="EmphasisTypeItalic">A</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>) that can be eventually modified by an external influx of molecules (<em xmlns="" class="EmphasisTypeItalic">A</em><sup><span class="stix">â</span></sup>) and a dilution protocol (see below). In our model we consider that signalling molecules degrade at the same rate whether they are cytoplasmic or not. Finally, we consider a <em xmlns="" class="EmphasisTypeItalic">DNA</em> duplication process. Such modelling scheme can be formally written as a set of chemical reactions:<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig1_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig1_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Scheme of the LuxI/LuxR regulatory network.</strong> The LuxR (<em class="EmphasisTypeItalic">R</em>) protein activates the operon upon binding to autoinducer molecules (<em class="EmphasisTypeItalic">A</em>). The <em class="EmphasisTypeItalic">lux01</em> operon lacks the <em class="EmphasisTypeItalic">luxI</em> gene and therefore cells cannot produce their own autoinducer and exogenous signalling molecules are needed to activate the expression of <em class="EmphasisTypeItalic">luxR</em> and GFP [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR10">10</a></span>]. On the other hand, the <em class="EmphasisTypeItalic">lux02</em> operon carries a <em class="EmphasisTypeItalic">luxI::gfp</em> fusion and allows for the production of autoinducer and self-induction (see text for details).</p>
                      </div></figcaption></figure></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div id="Equ1" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mtable columnalign="right center left" class="eqnarray">
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>Î±</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <msub>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>+</mo>
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>Î±</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                  <msub>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>+</mo>
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxI</mtext>
                                  <mo>:</mo>
                                  <mo>:</mo>
                                  <mtext mathvariant="italic">gfp</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxI</mtext>
                                  <mo>:</mo>
                                  <mo>:</mo>
                                  <mtext mathvariant="italic">gfp</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>p</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mtext mathvariant="italic">luxR</mtext>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxI</mtext>
                                  <mo>:</mo>
                                  <mo>:</mo>
                                  <mtext mathvariant="italic">gfp</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>p</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxI</mtext>
                                  <mo>:</mo>
                                  <mo>:</mo>
                                  <mtext mathvariant="italic">gfp</mtext>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mtext mathvariant="italic">luxI</mtext>
                              <mo>:</mo>
                              <mo>:</mo>
                              <mtext mathvariant="italic">gfp</mtext>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">luxI</mtext>
                              <mo>:</mo>
                              <mo>:</mo>
                              <mtext mathvariant="italic">gfp</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mi>A</mi>
                              <mo>+</mo>
                              <mtext mathvariant="italic">luxI</mtext>
                              <mo>:</mo>
                              <mo>:</mo>
                              <mtext mathvariant="italic">gfp</mtext>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">luxR</mtext>
                              <mo>+</mo>
                              <mi>A</mi>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <munder>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                    <mrow>
                                      <msubsup>
                                        <mrow>
                                          <mi>k</mi>
                                        </mrow>
                                        <mrow>
                                          <mn>1</mn>
                                        </mrow>
                                        <mrow>
                                          <mo>â</mo>
                                        </mrow>
                                      </msubsup>
                                    </mrow>
                                  </munder>
                                </mrow>
                                <mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>1</mn>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>/</mo>
                                  <msub>
                                    <mrow>
                                      <mi>K</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>d</mi>
                                      <mn>1</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">luxR</mtext>
                              <mo>Â·</mo>
                              <mi>A</mi>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mn>2</mn>
                              <mfenced separators="" open="(" close=")">
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                  <mo>Â·</mo>
                                  <mi>A</mi>
                                </mrow>
                              </mfenced>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <munder>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                    <mrow>
                                      <msubsup>
                                        <mrow>
                                          <mi>k</mi>
                                        </mrow>
                                        <mrow>
                                          <mn>2</mn>
                                        </mrow>
                                        <mrow>
                                          <mo>â</mo>
                                        </mrow>
                                      </msubsup>
                                    </mrow>
                                  </munder>
                                </mrow>
                                <mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>2</mn>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>/</mo>
                                  <msub>
                                    <mrow>
                                      <mi>K</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>d</mi>
                                      <mn>2</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mtext mathvariant="italic">DNA</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <munder>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                    <mrow>
                                      <msubsup>
                                        <mrow>
                                          <mi>k</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">lux</mtext>
                                        </mrow>
                                        <mrow>
                                          <mo>â</mo>
                                        </mrow>
                                      </msubsup>
                                    </mrow>
                                  </munder>
                                </mrow>
                                <mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>k</mi>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">lux</mtext>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>/</mo>
                                  <msub>
                                    <mrow>
                                      <mi>K</mi>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">dlux</mtext>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mi>A</mi>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <munder>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">rD</mtext>
                                    </mrow>
                                  </munder>
                                </mrow>
                                <mrow>
                                  <mi>D</mi>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">ext</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mi>A</mi>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">ext</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxR</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">mR</mtext>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                        </mtable>
                      </mrow></semantics></math></div></div>
                </div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div id="Equ2" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mtable columnalign="right center left" class="eqnarray">
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">mRN</mtext>
                              <msub>
                                <mrow>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">luxI</mtext>
                                  <mo>:</mo>
                                  <mo>:</mo>
                                  <mtext mathvariant="italic">gfp</mtext>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">mI</mtext>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">luxR</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">luxI</mtext>
                              <mo>:</mo>
                              <mo>:</mo>
                              <mtext mathvariant="italic">gfp</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>C</mi>
                                        </mrow>
                                        <mrow>
                                          <mn>2</mn>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">luxR</mtext>
                              <mo>Â·</mo>
                              <mi>A</mi>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>d</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>C</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mo>â</mo>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">ln</mtext>
                                  <mo>(</mo>
                                  <mn>2</mn>
                                  <mo>)</mo>
                                  <mo>/</mo>
                                  <mi>Ï</mi>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>+</mo>
                              <mtext mathvariant="italic">DNA</mtext>
                            </mtd>
                          </mtr>
                          <mtr>
                            <mtd class="eqnarray-1">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                            </mtd>
                            <mtd class="eqnarray-2">
                              <mover class="overset">
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">ln</mtext>
                                  <mo>(</mo>
                                  <mn>2</mn>
                                  <mo>)</mo>
                                  <mo>/</mo>
                                  <mi>Ï</mi>
                                </mrow>
                              </mover>
                            </mtd>
                            <mtd class="eqnarray-3">
                              <mtext mathvariant="italic">DNA</mtext>
                              <mo>Â·</mo>
                              <msub>
                                <mrow>
                                  <mfenced separators="" open="(" close=")">
                                    <mrow>
                                      <mtext mathvariant="italic">luxR</mtext>
                                      <mo>Â·</mo>
                                      <mi>A</mi>
                                    </mrow>
                                  </mfenced>
                                </mrow>
                                <mrow>
                                  <mn>2</mn>
                                </mrow>
                              </msub>
                              <mo>+</mo>
                              <mi>DNA.</mi>
                            </mtd>
                          </mtr>
                        </mtable>
                      </mrow></semantics></math></div> <div class="EquationNumber">(1)</div></div>
                </div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">As revealed by the set of reactions (1), we assume that the regulatory complex (<em xmlns="" class="EmphasisTypeItalic">luxR</em>Â·<em xmlns="" class="EmphasisTypeItalic">A</em>)<sub>2</sub> activates the transcription of <em xmlns="" class="EmphasisTypeItalic">luxI</em> and <em xmlns="" class="EmphasisTypeItalic">luxR</em> in opposite directions upon binding to the <em xmlns="" class="EmphasisTypeItalic">DNA</em>. These reactions account for the main regulatory interactions of both <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> constructs. Since <em xmlns="" class="EmphasisTypeItalic">lux01</em> lacks the <em xmlns="" class="EmphasisTypeItalic">luxI</em> gene the autoinducer, <em xmlns="" class="EmphasisTypeItalic">A</em>, cannot be synthesized, i.e. <em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sub>=0, and an exogenous supply of the signalling molecule is required to induce the system. The expression rates of <em xmlns="" class="EmphasisTypeItalic">luxI</em> and <em xmlns="" class="EmphasisTypeItalic">luxR</em> depend on the initiation rate of transcription, the speed of elongation, the length of the transcript, and the rate of translation and postmodification into functional proteins. We take into account the differences due to these intermediate processes in an effective manner by using different transcription/translation rates for the <em xmlns="" class="EmphasisTypeItalic">luxR</em> and <em xmlns="" class="EmphasisTypeItalic">luxI::gfp</em> genes. Note that we assume that there are basal transcriptional rates, <em xmlns="" class="EmphasisTypeItalic">Î±</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">Î±</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>, even though the regulatory complex (<em xmlns="" class="EmphasisTypeItalic">luxR</em>Â·<em xmlns="" class="EmphasisTypeItalic">A</em>)<sub>2</sub> is not bound to the promoter region of the <em xmlns="" class="EmphasisTypeItalic">DNA</em>. Still, since <em xmlns="" class="EmphasisTypeItalic">Î±</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub>,<em xmlns="" class="EmphasisTypeItalic">Î±</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub><span class="stix">âª</span>1 (see parameter values below), the maximum transcriptional rates take place when the activator complex is bound.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Deterministic and stochastic approaches: cell growth and division</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The equations (<span class="InternalRef"><a href="#Equ1">1</a></span>) lead to a Master equation description that can be sampled exactly by means of the Gillespie algorithm [<span class="CitationRef"><a href="#CR32">32</a></span>]. This approach is suitable for the characterization of the system at the single cell level. Complementary to this, if the number of molecules of the species is large enough such that the fluctuations can be neglected, a set of ordinary differential equations (ODEs) can be derived from Eqs. (<span class="InternalRef"><a href="#Equ1">1</a></span>) (see Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Text S1). The ODEs formalism is then appropriate to account for the behaviour at the colony level since noise averages out in that case. Herein we make use of both stochastic and deterministic descriptions as follows. As for the deterministic model, we consider that all cells share their cytoplasm in a <em xmlns="" class="EmphasisTypeItalic">single</em> volume <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>(Figure <span class="InternalRef"><a href="#Fig2">2</a></span>). Chemical species <em xmlns="" class="EmphasisTypeItalic">X</em> inside <em xmlns="" class="EmphasisTypeItalic">the</em> cell are described by their concentration, <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">X</em>
                  </sub>, in <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>. Therefore, this model can only be used to study the dynamics of species averaged over all the cells in the population. From an experimental point of view, the population average can be measured determining the average bulk fluorescence of the <em xmlns="" class="EmphasisTypeItalic">GFP</em> reporter of the cell culture by means of a fluorometer or by averaging the fluorescence data obtained with a flow cytometer.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig2_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig2_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 2</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Scheme of the deterministic and stochastic modelling approaches. A</strong>: In the deterministic model, the population of cells is described by a unique volume with average and continuous concentrations of all species, including the DNA carrying the QS network (small circles). Cellular growth is also taken into account in this approach. <strong class="EmphasisTypeBold">B</strong>: In the stochastic model, cells are modelled as individual compartments that can grow and divide and all molecular species are represented as discrete entities. In both cases, <strong class="EmphasisTypeBold">A</strong> and <strong class="EmphasisTypeBold">B</strong>, we assume that all species are well-stirred inside the cells and in the medium. In order to maintain a constant cell density, as in the experiments we aim to model, we implement a dilution protocol. In the deterministic model the dilution removes continuously cytoplasmic material in order to compensate the cell growth. In the stochastic model individual cells are removed every time a new cell is born (see Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM2">2</a></span>: Video S1).</p>
                      </div></figcaption></figure></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">We notice that our <em xmlns="" class="EmphasisTypeItalic">in silico</em> experiments span up to 100 hours of cell culture growth in some cases (simulated experimental time, not computational time). Thus, regardless of the description, and in addition to the dynamics of the regulatory network, we also need to take into account the effects of cell growth. If cells are maintained in the exponential phase with doubling time <em xmlns="" class="EmphasisTypeItalic">Ï</em> then the dynamics of the volume of <em xmlns="" class="EmphasisTypeItalic">the</em> cell is <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>(<em xmlns="" class="EmphasisTypeItalic">t</em>)=<em xmlns="" class="EmphasisTypeItalic">V</em><sub>0,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>2<sup><em xmlns="" class="EmphasisTypeItalic">t</em>/<em xmlns="" class="EmphasisTypeItalic">Ï</em></sup>. Where <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>=<em xmlns="" class="EmphasisTypeItalic">N</em> <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>, <em xmlns="" class="EmphasisTypeItalic">N</em> being the number of cells in the colony and <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>the volume of a single cell at the beginning of the cell cycle. As a consequence, the cellular growth introduces dilution terms, <span class="InlineEquation" id="IEq2"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mo>â</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>X</mi>
                          </mrow>
                        </msub>
                        <mfrac>
                          <mrow>
                            <mtext mathvariant="italic">ln</mtext>
                            <mo>(</mo>
                            <mn>2</mn>
                            <mo>)</mo>
                          </mrow>
                          <mrow>
                            <mi>Ï</mi>
                          </mrow>
                        </mfrac>
                      </mrow></semantics></math></span>, in the r.h.s. of the ODEs of all species, with the exception of the autoinducer in the medium <em xmlns="" class="EmphasisTypeItalic">A</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>. On the other hand, cell division events lead to the duplication of the genetic material. The latter is taken into account by adding the term <span class="InlineEquation" id="IEq3"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mo>+</mo>
                        <mfrac>
                          <mrow>
                            <mtext mathvariant="italic">ln</mtext>
                            <mo>(</mo>
                            <mn>2</mn>
                            <mo>)</mo>
                          </mrow>
                          <mrow>
                            <mi>Ï</mi>
                          </mrow>
                        </mfrac>
                        <mfenced separators="" open="(" close=")">
                          <mrow>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mtext mathvariant="italic">DNA</mtext>
                              </mrow>
                            </msub>
                            <mo>+</mo>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mtext mathvariant="italic">DNA</mtext>
                                <mo>Â·</mo>
                                <msub>
                                  <mrow>
                                    <mfenced separators="" open="(" close=")">
                                      <mrow>
                                        <mtext mathvariant="italic">luxR</mtext>
                                        <mo>Â·</mo>
                                        <mi>A</mi>
                                      </mrow>
                                    </mfenced>
                                  </mrow>
                                  <mrow>
                                    <mn>2</mn>
                                  </mrow>
                                </msub>
                              </mrow>
                            </msub>
                          </mrow>
                        </mfenced>
                      </mrow></semantics></math></span> to the ODE that describes the concentration of <em xmlns="" class="EmphasisTypeItalic">DNA</em>. This term compensates exactly for the cell growth dilution such that <span class="InlineEquation" id="IEq4"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mtext mathvariant="italic">DNA</mtext>
                            <mo>,</mo>
                            <mtext mathvariant="italic">tot</mtext>
                          </mrow>
                        </msub>
                        <mo>=</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mtext mathvariant="italic">DNA</mtext>
                          </mrow>
                        </msub>
                        <mo>+</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mtext mathvariant="italic">DNA</mtext>
                            <mo>Â·</mo>
                            <msub>
                              <mrow>
                                <mfenced separators="" open="(" close=")">
                                  <mrow>
                                    <mtext mathvariant="italic">luxR</mtext>
                                    <mo>Â·</mo>
                                    <mi>A</mi>
                                  </mrow>
                                </mfenced>
                              </mrow>
                              <mrow>
                                <mn>2</mn>
                              </mrow>
                            </msub>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span>, i.e. the total concentration of DNA, is kept constant.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">In our simulations, as in the experiments we aim to reproduce, the cell density is kept constant. This can be achieved by means of an external dilution protocol (see below) that compensates for cell proliferation. We then keep the volume <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub> constant and define the external volume, <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>, such that the total volume of the cell culture reads <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">tot</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub> + <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>. Accordingly, the parameter <em xmlns="" class="EmphasisTypeItalic">r</em>, see equations (<span class="InternalRef"><a href="#Equ1">1</a></span>), reads <em xmlns="" class="EmphasisTypeItalic">r</em>=<em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>/<em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>. We assume that molecules are homogeneously distributed inside both the cytoplasm and the external volume (i.e. spatial effects are disregarded). Finally, the resulting ODEs are numerically integrated.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">In order to study the role of noise in a population of cells communicating by QS, we build also a stochastic model of a population of bacteria. In this case, each bacterium is described as a single cell carrying a copy of the regulatory network. The ensemble of all the chemical reactions in all cells, including the diffusion reaction, are treated as one global system. We apply the Gillespie algorithm [<span class="CitationRef"><a href="#CR32">32</a></span>] to compute the time of the next reaction, choose the reaction channel from the list of all possible reactions and update the number of molecules according to the reaction stoichiometry. We model the system of cells as a global stochastic system in order to simulate as exactly as possible the stochastic dynamics of all chemical species, in particular that of autoinducer molecules. The noise in the signalling molecule originates from different sources: randomness in its synthesis by LuxI, fluctuations at the level of the number of molecules of LuxI, and randomness in the diffusion reaction of the autoinducer. The latter is particularly important since it leads to correlations between cells as follows. An autoinducer molecule can diffuse out of the cytoplasm of one cell into the medium, thereby increasing the number of molecules in the external volume by one; this increase in the level of <em xmlns="" class="EmphasisTypeItalic">A</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>changes the probabilities of an autoinducer molecule to diffuse into any other cell. Thus, all the cells are coupled through the diffusion reaction. We note that while a possible optimization of the algorithm relies on parallelizing the code such that each cell evolves independently [<span class="CitationRef"><a href="#CR25">25</a></span>], this approximation is prone to introduce errors in the dynamics of the signalling molecule because the aforementioned correlations are neglected.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">As mentioned above, cell growth introduces a dilution of the molecules in a cell. We implement cell growth in our stochastic model by allowing the volume of cell <em xmlns="" class="EmphasisTypeItalic">i</em> to change in time as,<div id="Equ3" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mi>V</mi>
                            </mrow>
                            <mrow>
                              <mi>c</mi>
                              <mo>,</mo>
                              <mi>i</mi>
                            </mrow>
                          </msub>
                          <mo>(</mo>
                          <mi>t</mi>
                          <mo>)</mo>
                          <mo>=</mo>
                          <msub>
                            <mrow>
                              <mi>V</mi>
                            </mrow>
                            <mrow>
                              <mn>0</mn>
                            </mrow>
                          </msub>
                          <msup>
                            <mrow>
                              <mn>2</mn>
                            </mrow>
                            <mrow>
                              <mi>t</mi>
                              <mo>/</mo>
                              <msub>
                                <mrow>
                                  <mi>Ï</mi>
                                </mrow>
                                <mrow>
                                  <mi>i</mi>
                                </mrow>
                              </msub>
                            </mrow>
                          </msup>
                          <mo>,</mo>
                        </mrow>
                      </mrow></semantics></math></div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>is the volume of a cell at the beginning of the cell cycle (same for all cells), <em xmlns="" class="EmphasisTypeItalic">Ï</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>is the duration of the cell cycle of cell <em xmlns="" class="EmphasisTypeItalic">i</em>, and <em xmlns="" class="EmphasisTypeItalic">t</em> is referred to the precedent division event. When <em xmlns="" class="EmphasisTypeItalic">t</em>=<em xmlns="" class="EmphasisTypeItalic">Ï</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub> the cell <em xmlns="" class="EmphasisTypeItalic">i</em> has doubled its volume and a new division takes place. At this time the internal clocks and volumes of daughter cells are reset to zero and <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>respectively. Moreover, when a cell divides, proteins, mRNAs and signalling molecules are binomially distributed [<span class="CitationRef"><a href="#CR33">33</a></span>] between daughter cells and one copy of the DNA is given to each cell. We note that regulatory complexes bound to the DNA are detached prior to the distribution between daughter cells. As in the case of the deterministic model, we assume that the cell density is maintained constant during experiments due to a compensational external efflux that wash away cells in the culture (see below). In relation to the effect of the cell volume of individual cells on the diffusion rate of the autoinducer, we note that in this case,<div id="Equ4" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mi>r</mi>
                            </mrow>
                            <mrow>
                              <mi>i</mi>
                            </mrow>
                          </msub>
                          <mfenced open="(" close=")">
                            <mi>t</mi>
                          </mfenced>
                          <mo>=</mo>
                          <mfrac>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>V</mi>
                                </mrow>
                                <mrow>
                                  <mi>c</mi>
                                  <mo>,</mo>
                                  <mi>i</mi>
                                </mrow>
                              </msub>
                              <mo>(</mo>
                              <mi>t</mi>
                              <mo>)</mo>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>V</mi>
                                </mrow>
                                <mrow>
                                  <mtext mathvariant="italic">tot</mtext>
                                </mrow>
                              </msub>
                              <mo>â</mo>
                              <munderover accentunder="false" accent="false">
                                <mrow>
                                  <mo mathsize="big">â</mo>
                                </mrow>
                                <mrow>
                                  <mi>j</mi>
                                  <mo>=</mo>
                                  <mn>1</mn>
                                </mrow>
                                <mrow>
                                  <mi>N</mi>
                                </mrow>
                              </munderover>
                              <msub>
                                <mrow>
                                  <mi>V</mi>
                                </mrow>
                                <mrow>
                                  <mi>c</mi>
                                  <mo>,</mo>
                                  <mi>j</mi>
                                </mrow>
                              </msub>
                              <mo>(</mo>
                              <mi>t</mi>
                              <mo>)</mo>
                            </mrow>
                          </mfrac>
                          <mi>.</mi>
                        </mrow>
                      </mrow></semantics></math></div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The duration of the cell cycle, <em xmlns="" class="EmphasisTypeItalic">Ï</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>is different for each cell and is set independently after a division according to the following stochastic rule [<span class="CitationRef"><a href="#CR34">34</a></span>],<div id="Equ5" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mi>Ï</mi>
                            </mrow>
                            <mrow>
                              <mi>i</mi>
                            </mrow>
                          </msub>
                          <mo>=</mo>
                          <mi>Î»Ï</mi>
                          <mo>+</mo>
                          <mfenced separators="" open="(" close=")">
                            <mrow>
                              <mn>1</mn>
                              <mo>â</mo>
                              <mi>Î»</mi>
                            </mrow>
                          </mfenced>
                          <mover accent="true">
                            <mrow>
                              <mi>Ï</mi>
                            </mrow>
                            <mo>~</mo>
                          </mover>
                          <mo>,</mo>
                        </mrow>
                      </mrow></semantics></math></div></div></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">Ï</em> and <span class="InlineEquation" id="IEq5"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi>Ï</mi>
                          </mrow>
                          <mo>~</mo>
                        </mover>
                      </mrow></semantics></math></span> denote, respectively, the deterministic and stochastic components of the cell cycle duration, and <em xmlns="" class="EmphasisTypeItalic">Î»</em><span class="stix">â</span>[0,1] is a parameter that weights their relative importance. The stochastic component accounts for the period of time between events driven by a Poissonian process and satisfies an exponential distribution,<div id="Equ6" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi>Ï</mi>
                          <mfenced separators="" open="(" close=")">
                            <mrow>
                              <mover accent="true">
                                <mrow>
                                  <mi>Ï</mi>
                                </mrow>
                                <mo>~</mo>
                              </mover>
                            </mrow>
                          </mfenced>
                          <mo>=</mo>
                          <mfrac>
                            <mrow>
                              <msup>
                                <mrow>
                                  <mi>e</mi>
                                </mrow>
                                <mrow>
                                  <mo>â</mo>
                                  <mfrac>
                                    <mrow>
                                      <mover accent="true">
                                        <mrow>
                                          <mi>Ï</mi>
                                        </mrow>
                                        <mo>~</mo>
                                      </mover>
                                    </mrow>
                                    <mrow>
                                      <mi>Ï</mi>
                                    </mrow>
                                  </mfrac>
                                </mrow>
                              </msup>
                            </mrow>
                            <mrow>
                              <mi>Ï</mi>
                            </mrow>
                          </mfrac>
                          <mi>.</mi>
                        </mrow>
                      </mrow></semantics></math></div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">In this way, we allow variability from cell to cell in regards of the duration of the cell cycle, yet setting a minimum cell cycle duration, <em xmlns="" class="EmphasisTypeItalic">Î»Ï</em>. According to these definitions, the average duration and standard deviation of the cell cycle are <em xmlns="" class="EmphasisTypeItalic">Ï</em> and (1â<em xmlns="" class="EmphasisTypeItalic">Î»</em>)<em xmlns="" class="EmphasisTypeItalic">Ï</em> respectively.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Finally, we notice that in principle the Gillespie algorithm needs to be adapted in order to take into account the time-dependent cell volume. The propensity of a second-order reaction at cell <em xmlns="" class="EmphasisTypeItalic">i</em> at time <em xmlns="" class="EmphasisTypeItalic">t</em> scales as <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>(<em xmlns="" class="EmphasisTypeItalic">t</em>)=<em xmlns="" class="EmphasisTypeItalic">p</em><sub>0</sub><em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>/<em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">i</em></sub>(<em xmlns="" class="EmphasisTypeItalic">t</em>), where <em xmlns="" class="EmphasisTypeItalic">p</em><sub>0</sub> stands for propensity of the reaction at division time when <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">i</em></sub>(0)=<em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>. The propensity <em xmlns="" class="EmphasisTypeItalic">p</em><sub>0</sub>are derived from the corresponding reaction rate, <em xmlns="" class="EmphasisTypeItalic">k,</em> by dividing the latter by the initial cell volume, <em xmlns="" class="EmphasisTypeItalic">p</em><sub>0</sub>=<em xmlns="" class="EmphasisTypeItalic">k</em>/<em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>. In addition to the change in the propensities of the reaction channels, the algorithm would also need to be adapted to compute the time till next reaction [<span class="CitationRef"><a href="#CR35">35</a></span>]. However, in our case, since all reactions rates are faster than the rate of variation of the cell volume, <span class="stix">â¼</span>1/<em xmlns="" class="EmphasisTypeItalic">Ï</em>, (see parameter values below) then the volume increase is negligible during the time interval until the next reaction takes place. Consequently, we can adiabatically eliminate the volume growth dynamics and safely assume that the volume-dependent propensities remain constant until the next reaction occurs. Summarizing, at a given time <em xmlns="" class="EmphasisTypeItalic">t</em> we compute, as described above, the time-dependent propensities based on the volume of the cell at that time and, according to those, we determine the time at which the next reaction takes place, <em xmlns="" class="EmphasisTypeItalic">t</em> + <span class="stix">â³</span><em xmlns="" class="EmphasisTypeItalic">t</em>, following the standard Gillespie algorithm.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Gene expression noise: burst size</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">During translation mRNA molecules are translated into proteins following a bursting dynamics [<span class="CitationRef"><a href="#CR36">36</a></span>â<span class="CitationRef"><a href="#CR38">38</a></span>]. The so-called burst size, <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">X</em>
                  </sub>, is defined as the ratio between the protein <em xmlns="" class="EmphasisTypeItalic">X</em> production rate and the mRNA <em xmlns="" class="EmphasisTypeItalic">X</em> degradation rate. It has been shown that <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">X</em>
                  </sub> is directly related to the intensity of gene expression noise [<span class="CitationRef"><a href="#CR36">36</a></span>, <span class="CitationRef"><a href="#CR39">39</a></span>]. Thus, for the same average protein concentration, the larger <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">X</em>
                  </sub> is, the more fluctuating expression dynamics is displayed by protein <em xmlns="" class="EmphasisTypeItalic">X</em>. In our stochastic simulations we use the burst size <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">X</em>
                  </sub>as a parameter to tune the noise intensity at the level of <em xmlns="" class="EmphasisTypeItalic">luxI</em> and <em xmlns="" class="EmphasisTypeItalic">luxR</em> and study its effects. Unless explicitly indicated otherwise, the bursting size in the stochastic simulations is <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>=20.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">External dilution protocol</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">In controlled experimental setups it is advantageous to keep the cell density constant. This is carried out by means of an external dilution protocol that compensates for cell growth. Experimentally, this is usually achieved by periodic dilutions of the cell culture [<span class="CitationRef"><a href="#CR10">10</a></span>] or by a continuous flow of liquid medium in a chemostat or in a microfluidic device [<span class="CitationRef"><a href="#CR40">40</a></span>]. This procedure allows to measure the stationary concentration of the signalling molecule at a given cell density and/or to estimate the threshold of the QS collective response of a cell culture. Moreover, the external dilution is also important in order to maintain cells in the exponential growth phase and prevent depletion of nutrients in the medium. Additionally, the levels of the autoinducer can be controlled by adding/removing exogenous signalling molecules in/from the culture buffer. We implement those in our simulations as follows.</p>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">In the deterministic model, as shown in Figure <span class="InternalRef"><a href="#Fig2">2</a></span>, we assume a unique cell with volume <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>. Cell density is controlled by a continuous efflux that removes cytoplasm and culture medium at a rate that compensates exactly for the cell growth, such that the volume <em xmlns="" class="EmphasisTypeItalic">V</em><sub><em xmlns="" class="EmphasisTypeItalic">c</em>,<em xmlns="" class="EmphasisTypeItalic">tot</em></sub>remains constant. Concurrently, a continuous influx of equal and opposite rate brings fresh medium to the cell culture. In our <em xmlns="" class="EmphasisTypeItalic">in silico</em> stochastic experiments, the efflux is reproduced by removing molecules, <em xmlns="" class="EmphasisTypeItalic">A</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>, from the medium and washing away cells by âdeletingâ a cell picked at random in the population each time a new cell is born.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">In our simulations, as in the experiments we aim to reproduce, the exogenous autoinducer concentration <span class="InlineEquation" id="IEq6"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span> is the control parameter [<span class="CitationRef"><a href="#CR10">10</a></span>]. This means that the levels of autoinducer are controlled by varying the concentration of exogenous autoinducer in the dilution buffer (influx). The influx of exogenous autoinducer molecules, together with the efflux of culture medium, can be represented by the following reaction,<div id="Equ7" class="Equation EquationMathjax"><div class="EquationContent"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mi>A</mi>
                            </mrow>
                            <mrow>
                              <mtext mathvariant="italic">ext</mtext>
                            </mrow>
                          </msub>
                          <mover class="overset">
                            <mrow>
                              <munder>
                                <mrow>
                                  <mo>â</mo>
                                </mrow>
                                <mrow>
                                  <mi>Î³</mi>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <msub>
                                    <mrow>
                                      <mi>V</mi>
                                    </mrow>
                                    <mrow>
                                      <mtext mathvariant="italic">tot</mtext>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </munder>
                            </mrow>
                            <mrow>
                              <mi>Î³</mi>
                            </mrow>
                          </mover>
                          <mo>â</mo>
                          <mi>.</mi>
                        </mrow>
                      </mrow></semantics></math></div></div></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">Î³</em>=<em xmlns="" class="EmphasisTypeItalic">ln</em>(2)/<em xmlns="" class="EmphasisTypeItalic">Ï</em>. That is, an efflux removes autoinducer molecules from the external volume at a rate <em xmlns="" class="EmphasisTypeItalic">Î³</em> and an influx introduces signalling molecules in the external volume at a rate <span class="InlineEquation" id="IEq7"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mi>Î³</mi>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <msub>
                          <mrow>
                            <mi>V</mi>
                          </mrow>
                          <mrow>
                            <mtext mathvariant="italic">tot</mtext>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span>. In the deterministic description, the last equation leads to an additional term at the r.h.s. of the ODE for the concentration of <em xmlns="" class="EmphasisTypeItalic">A</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>:<span class="InlineEquation" id="IEq8"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mo>+</mo>
                        <mi>Î³</mi>
                        <mfenced separators="" open="(" close=")">
                          <mrow>
                            <msubsup>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msubsup>
                            <mfrac>
                              <mrow>
                                <msub>
                                  <mrow>
                                    <mi>V</mi>
                                  </mrow>
                                  <mrow>
                                    <mtext mathvariant="italic">tot</mtext>
                                  </mrow>
                                </msub>
                              </mrow>
                              <mrow>
                                <msub>
                                  <mrow>
                                    <mi>V</mi>
                                  </mrow>
                                  <mrow>
                                    <mtext mathvariant="italic">ext</mtext>
                                  </mrow>
                                </msub>
                              </mrow>
                            </mfrac>
                            <mo>â</mo>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <msub>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mtext mathvariant="italic">ext</mtext>
                                  </mrow>
                                </msub>
                              </mrow>
                            </msub>
                          </mrow>
                        </mfenced>
                      </mrow></semantics></math></span>. We notice that in our simulations, as in experiments, <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">tot</em>
                  </sub>/<em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub><span class="stix">â</span>1. In the absence of synthesis (e.g. <em xmlns="" class="EmphasisTypeItalic">lux01</em>) and taking into account that the degradation is slower than the diffusion and the influx rate, it is easy to see that the concentration of autoinducer, both inside and outside <em xmlns="" class="EmphasisTypeItalic">the</em> cell, tends to <span class="InlineEquation" id="IEq9"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span>: the desired control value of the autoinducer concentration (see Additional file <span class="InternalRef"><a href="#MOESM3">3</a></span>: Figure S1).</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Parameters</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The parameters used in our model are listed in Table <span class="InternalRef"><a href="#Tab1">1</a></span>. When possible, parameter values are fixed or estimated by using experimental measurements found in the literature. The rest of the parameters are fitted to the experimental data of [<span class="CitationRef"><a href="#CR10">10</a></span>] using the deterministic model to reproduce the main characteristics of the response curves of the <em xmlns="" class="EmphasisTypeItalic">lux01</em> operon: a difference of two orders of magnitude in the level of expression of GFP between the low and the high states, a hysteresis effect in the range of autoinducer concentrations <span class="InlineEquation" id="IEq16"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mn>0</mn>
                        <mo>&lt;</mo>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>&lt;</mo>
                        <mn>15</mn>
                        <mspace width="1em"></mspace>
                        <mtext mathvariant="italic">nM</mtext>
                      </mrow></semantics></math></span>, and a time to reach steady-state at full induction (<span class="InlineEquation" id="IEq17"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>=</mo>
                        <mn>100</mn>
                        <mspace width="1em"></mspace>
                        <mtext mathvariant="italic">nM</mtext>
                      </mrow></semantics></math></span>) shorter than 6 hours. In regards of the cell density, based on an estimate of the CFU/mL for an average OD of 0.5 for <em xmlns="" class="EmphasisTypeItalic">E. coli</em> cells, we take a typical value of <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>=5Â·10<sup>8</sup>cells/mL. Moreover, in order to keep the computational time within reasonable limits, we choose a system size of <em xmlns="" class="EmphasisTypeItalic">N</em>=100 cells. After fixing the number of cells and the cell density, the total and external volumes are then respectively derived from the relations <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">N</em>/<em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">tot</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ext</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">tot</em>
                  </sub>â<em xmlns="" class="EmphasisTypeItalic">N</em> <em xmlns="" class="EmphasisTypeItalic">V</em><sub>0</sub>, where <em xmlns="" class="EmphasisTypeItalic">V</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">tot</em>
                  </sub>=2Â·10<sup>â4</sup><em xmlns="" class="EmphasisTypeItalic">Î¼L</em>. Finally, for the case of the <em xmlns="" class="EmphasisTypeItalic">lux02</em> operon there is one additional parameter that needs to be calibrated: the synthesis rate of the autoinducer, <em xmlns="" class="EmphasisTypeItalic">k</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sub>. The latter is adjusted such that the lower bound of the hysteresis region extends up to <span class="InlineEquation" id="IEq18"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>=</mo>
                        <mn>0</mn>
                      </mrow></semantics></math></span> as experimentally reported.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">
                          <strong class="EmphasisTypeBold">Parameters used in the deterministic and stochastic simulations</strong>
                        </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><colgroup><col align="left"></colgroup><thead><tr class="header"><th align="left"><p xmlns="" class="SimplePara">Parameter</p></th><th align="left"><p xmlns="" class="SimplePara">Description</p></th><th align="left"><p xmlns="" class="SimplePara">Value</p></th><th align="left"><p xmlns="" class="SimplePara">Reference</p></th></tr></thead><tbody><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">K</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">d</em> 1</sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">dissociation constant of LuxR to A</p></td><td align="left"><p xmlns="" class="SimplePara">100 <em class="EmphasisTypeItalic">nM</em></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR41">41</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq19"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                    <msubsup>
                                      <mrow>
                                        <mi>k</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>1</mn>
                                      </mrow>
                                      <mrow>
                                        <mo>â</mo>
                                      </mrow>
                                    </msubsup>
                                  </mrow></semantics></math></span>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">unbinding rate of LuxR to A</p></td><td align="left"><p xmlns="" class="SimplePara">10 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">K</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml"><em xmlns="" class="EmphasisTypeItalic">d</em> 2</sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">dissociation constant of <em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em> dimerization</p></td><td align="left"><p xmlns="" class="SimplePara">20 <em class="EmphasisTypeItalic">nM</em></p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq20"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                    <msubsup>
                                      <mrow>
                                        <mi>K</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>2</mn>
                                      </mrow>
                                      <mrow>
                                        <mo>â</mo>
                                      </mrow>
                                    </msubsup>
                                  </mrow></semantics></math></span>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">dissociation rate of dimer (<em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em>)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub></p></td><td align="left"><p xmlns="" class="SimplePara">1 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">k</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">A</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">synthesis rate of A by LuxI</p></td><td align="left"><p xmlns="" class="SimplePara">0.04 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">K</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">dlux</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">dissociation constant of (<em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em>)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub> to the lux promoter</p></td><td align="left"><p xmlns="" class="SimplePara">200 <em class="EmphasisTypeItalic">nM</em></p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq21"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                    <msubsup>
                                      <mrow>
                                        <mi>k</mi>
                                      </mrow>
                                      <mrow>
                                        <mtext mathvariant="italic">lux</mtext>
                                      </mrow>
                                      <mrow>
                                        <mo>â</mo>
                                      </mrow>
                                    </msubsup>
                                  </mrow></semantics></math></span>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">dissociation rate of (<em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em>)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub> to the lux promoter</p></td><td align="left"><p xmlns="" class="SimplePara">10 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">b</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">burst size</p></td><td align="left"><p xmlns="" class="SimplePara">20</p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR38">38</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">k</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">R</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">transcription rate of <em class="EmphasisTypeItalic">luxR</em></p></td><td align="left"><p xmlns="" class="SimplePara">200/<em class="EmphasisTypeItalic">b</em> <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">k</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">I</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">transcription rate of <em class="EmphasisTypeItalic">luxI</em></p></td><td align="left"><p xmlns="" class="SimplePara">50/<em class="EmphasisTypeItalic">b</em> <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">p</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">R</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">translation rate of <em class="EmphasisTypeItalic">luxR</em> mRNA</p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">b</em>
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">mR</em>
                              </sub>
                              <em class="EmphasisTypeItalic">mi</em>
                              <em class="EmphasisTypeItalic">n</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">â1</sup>
                            </p></td><td align="left">&nbsp;</td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">p</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">I</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">translation rate of <em class="EmphasisTypeItalic">luxI</em> mRNA</p></td><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">b</em>
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">mI</em>
                              </sub>
                              <em class="EmphasisTypeItalic">mi</em>
                              <em class="EmphasisTypeItalic">n</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">â1</sup>
                            </p></td><td align="left">&nbsp;</td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Î±</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">R</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">ratio between unactivated and activated rate of expression of <em class="EmphasisTypeItalic">luxR</em></p></td><td align="left"><p xmlns="" class="SimplePara">0.001</p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Î±</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">I</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">ratio between unactivated and activated rate of expression of <em class="EmphasisTypeItalic">luxI</em></p></td><td align="left"><p xmlns="" class="SimplePara">0.01</p></td><td align="left"><p xmlns="" class="SimplePara">fitted</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">A</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of <em class="EmphasisTypeItalic">A</em> (same inside and outside the cell)</p></td><td align="left"><p xmlns="" class="SimplePara">0.001 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR42">42</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq22"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                    <msub>
                                      <mrow>
                                        <mi>d</mi>
                                      </mrow>
                                      <mrow>
                                        <msub>
                                          <mrow>
                                            <mi>C</mi>
                                          </mrow>
                                          <mrow>
                                            <mn>2</mn>
                                          </mrow>
                                        </msub>
                                      </mrow>
                                    </msub>
                                  </mrow></semantics></math></span>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of (<em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em>)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub></p></td><td align="left"><p xmlns="" class="SimplePara">0.002 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">C</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of <em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em></p></td><td align="left"><p xmlns="" class="SimplePara">0.002 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">R</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of LuxR</p></td><td align="left"><p xmlns="" class="SimplePara">0.002 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">I</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of LuxI</p></td><td align="left"><p xmlns="" class="SimplePara">0.01 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">estimated</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">mR</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of <em class="EmphasisTypeItalic">luxR</em> mRNA</p></td><td align="left"><p xmlns="" class="SimplePara">0.347 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR43">43</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">d</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">mI</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">degradation rate of <em class="EmphasisTypeItalic">luxI</em> mRNA</p></td><td align="left"><p xmlns="" class="SimplePara">0.347 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR43">43</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">D</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">effective diffusion rate of <em class="EmphasisTypeItalic">A</em> through the cell membrane</p></td><td align="left"><p xmlns="" class="SimplePara">10 <em class="EmphasisTypeItalic">mi</em> <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">â1</sup></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR44">44</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Ï</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">cell cycle duration (doubling time) in RM/succinate at 30 C</p></td><td align="left"><p xmlns="" class="SimplePara">45 <em class="EmphasisTypeItalic">min</em></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR10">10</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Î»</em>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">relative weight between the det./sto. components of the cell cycle</p></td><td align="left"><p xmlns="" class="SimplePara">0.8</p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR33">33</a></span>, <span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR45">45</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">V</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">0</sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">cell volume at the beginning of cell cycle</p></td><td align="left"><p xmlns="" class="SimplePara">1.5 <em class="EmphasisTypeItalic">Î¼</em> <em class="EmphasisTypeItalic">m</em><sup xmlns="http://www.w3.org/1999/xhtml">3</sup></p></td><td align="left"><p xmlns="" class="SimplePara">[<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR46">46</a></span>]</p></td></tr><tr class="noclass"><td align="left"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">V</em>
                              <sub xmlns="http://www.w3.org/1999/xhtml">
                                <em xmlns="" class="EmphasisTypeItalic">tot</em>
                              </sub>
                            </p></td><td align="left"><p xmlns="" class="SimplePara">total cell culture volume</p></td><td align="left"><p xmlns="" class="SimplePara">2Â·10<sup xmlns="http://www.w3.org/1999/xhtml">â4</sup><em class="EmphasisTypeItalic">Î¼l</em></p></td><td align="left">&nbsp;</td></tr></tbody></table></div></figure></div>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">First passage time analysis</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">The mean first passage time at a given autoinducer concentration quantifies the average time that a cell takes to get activated or deactivated. For computing the first passage time in transitions, from low (high) to high (low) state, we take a single cell at the low (high) state and follow its dynamics until the GFP expression level reaches the high (low) state. We point out that the maximum GFP concentration refers to that of the deterministic simulations. In order to get enough statistics, we repeat this procedure, departing from the same initial condition, 10<sup>3</sup> times for each concentration of autoinducer.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec9" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Results</h2><div class="js-CollapseSection"><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">The deterministic model reproduces the experimental observations at the population level</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">The chemical kinetics formalism leads to a set of ODEs that describes the population average dynamics in terms of the concentration of the different species considered in our model (see Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>: Text S1). As in some experiments [<span class="CitationRef"><a href="#CR10">10</a></span>], we assume that the cell culture grows in an environment where the concentration of the external autoinducer in the medium, <span class="InlineEquation" id="IEq23"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msub>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mtext mathvariant="italic">ext</mtext>
                              </mrow>
                            </msub>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span>, is kept fixed and under well-stirred conditions. In addition, we implement a dilution protocol that compensates for cell growth and maintains the cell density constant (see Methods). We notice that in some experimental setups, e.g. [<span class="CitationRef"><a href="#CR10">10</a></span>], a periodic dilution protocol is applied for keeping the cell density constant; in our model, we keep the cell density constant by means of a continuous influx and efflux of culture medium, as in a chemostat or microfluidic device.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">We use the deterministic simulations as a benchmark of the regulatory interactions included in our model and also to fit/estimate some parameters such that the experimental data are reproduced (see [<span class="CitationRef"><a href="#CR10">10</a></span>]). Thus, by integrating numerically the rate equations derived from the population-averaged model, we compute the steady state concentration (induction time 100 hours) of GFP (<em xmlns="" class="EmphasisTypeItalic">lux01</em>) and LuxI::GFP (<em xmlns="" class="EmphasisTypeItalic">lux02</em>) as a function of <span class="InlineEquation" id="IEq24"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span>. The steady-state induction curves for increasing and decreasing autoinducer concentration of the <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> constructs are shown in Figure <span class="InternalRef"><a href="#Fig3">3</a></span>. We are able to reproduce the behaviour of the network at the steady-state, in particular a region of bistability in the range of autoinducer concentration <span class="InlineEquation" id="IEq25"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mn>2</mn>
                        <mtext>nM</mtext>
                        <mo>&lt;</mo>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>&lt;</mo>
                        <mn>15</mn>
                      </mrow></semantics></math></span> nM (<em xmlns="" class="EmphasisTypeItalic">lux01</em>) and <span class="InlineEquation" id="IEq26"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mn>0</mn>
                        <mtext>nM</mtext>
                        <mo>&lt;</mo>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>&lt;</mo>
                        <mn>15</mn>
                      </mrow></semantics></math></span> nM (<em xmlns="" class="EmphasisTypeItalic">lux02</em>). As shown by Williams et al., the <em xmlns="" class="EmphasisTypeItalic">luxR</em> regulation of the <em xmlns="" class="EmphasisTypeItalic">lux01</em> operon alone (positive feedback loop) is enough to yield a bistable response. Moreover, expression of LuxI in the <em xmlns="" class="EmphasisTypeItalic">lux02</em> operon restores the autoinduction loop and extends the lower bound of the hysteresis range to zero concentration of exogenous autoinducer as seen experimentally, indicating that once the operon is fully activated and cells produce their own autoinducer that increases the stability of the high state.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig3_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig3_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 3</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Response curves to autoinducer induction in the population-average model.</strong> <em class="EmphasisTypeItalic">lux01</em> (<strong class="EmphasisTypeBold">A</strong>) and <em class="EmphasisTypeItalic">lux02</em> (<strong class="EmphasisTypeBold">B</strong>) operons. The normalized GFP concentration is plotted as a function of the exogenous autoinducer concentration <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq27"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span>: steady-state response for increasing (arrow-free upper blue curve) and decreasing (arrow-free red curve) autoinducer concentration, response under 10 h induction time for increasing (blue curve with arrow) autoinducer concentration, transient response after 2 hours of induction (lower blue curve) from initially non-induced cells, decreasing-concentration trajectories (green curves) for cells weakly induced (2 hours) at <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq28"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>=</mo>
                                <mn>100</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span>, 75 nM and 50 nM, and decreasing-concentration trajectories (red curve with arrow) for cells fully induced (10 hours) at <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq29"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>=</mo>
                                <mn>100</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span>. The decreasing-concentration trajectories reduce the value of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq30"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> hourly by 25% (similar to the experiments in [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR10">10</a></span>]). The gray-shaded region between the increasing and decreasing steady-state curves reveals bistability in the range <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq31"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <mn>2</mn>
                                <mtext>nM</mtext>
                                <mo>&lt;</mo>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>&lt;</mo>
                                <mn>15</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span> (<em class="EmphasisTypeItalic">lux01</em>) and <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq32"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <mn>0</mn>
                                <mtext>nM</mtext>
                                <mo>&lt;</mo>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>&lt;</mo>
                                <mn>15</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span> (<em class="EmphasisTypeItalic">lux02</em>).</p>
                      </div></figcaption></figure></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Further simulations to check if the dynamics of our model is compatible with the experimental data refer to the behaviour of the system under non-stationary induction conditions and to the serial dilution protocol of the external medium [<span class="CitationRef"><a href="#CR10">10</a></span>]. As for the first, when cells are induced for 10 h, we observe that the bistability region increases (see Figure <span class="InternalRef"><a href="#Fig3">3</a></span>). As for the second, cells are partially induced at a fixed autoinducer concentration for 2 hours and afterwards the external medium is changed hourly to decrease the concentration of the autoinducer. In this case, the transient response of the cells (Figure <span class="InternalRef"><a href="#Fig3">3</a></span>, green curves) also reproduces the experimental observations. That is, from the point of view of the population average, the deterministic model is not only capable of reproducing the steady-state of the network but also its dynamics. Moreover, in agreement with experiments (see Figure S6 in [<span class="CitationRef"><a href="#CR10">10</a></span>]) our simulations reveal that the temporal scale for reaching a steady-state is much larger than the cell cycle duration. In order to clarify how noise and the induction time modifies the timing for the transition at the single cell level we then perform stochastic simulations.</p>
              </section><section id="Sec11" class="Section2 RenderAsSection2">
                <h3 class="Heading">The stochastic simulations reveal the interplay between non-stationary effects and noise</h3>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">Cells are subjected to intrinsic noise at the level of the mRNAs, regulatory proteins, i.e. LuxR and LuxI, and at the level of signalling molecules. In order to analyze the behaviour of individual cells and reveal how noise affects the QS switch, we perform stochastic simulations of a population of growing and dividing cells as described in the Methods section (see Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>: Video S1). The transition of an individual cell from the low to the high state, and the other way around, is intrinsically random and depends, among others, on the levels of autoinducer. Thus, inside a population some cells will jump while others remain in their current state leading to a bimodal phenotypic distribution. We compute the proportion of cells that are below and above a threshold of GFP equal to half-maximum GFP concentration. We consider the distribution of cells to be bimodal when the proportion of cells in either the low or the high state is below 90% and according to this we define the range of autoinducer concentration <span class="InlineEquation" id="IEq33"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mo>[</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msubsup>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mi>b</mi>
                                <mn>1</mn>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msubsup>
                          </mrow>
                        </msub>
                        <mo>,</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msubsup>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mi>b</mi>
                                <mn>2</mn>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msubsup>
                          </mrow>
                        </msub>
                        <mo>]</mo>
                      </mrow></semantics></math></span> for which there is bimodality. For low concentrations of autoinducer, <span class="InlineEquation" id="IEq34"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>&lt;</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msubsup>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mi>b</mi>
                                <mn>1</mn>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msubsup>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span>, the collective response of the cell population is unactivated, and for high concentrations, <span class="InlineEquation" id="IEq35"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>&gt;</mo>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msubsup>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mi>b</mi>
                                <mn>2</mn>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msubsup>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span>, such response activates most of the cells leading to a global response of the colony. On the other hand, within the bimodality range, the response is distributed between two subpopulations, thus failing to achieve a global coordination in the colony. In order to characterize this behaviour, we introduce the concept of precision in the QS switch as the inverse of the <span class="InlineEquation" id="IEq36"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msub>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <msub>
                              <mrow>
                                <mi>A</mi>
                              </mrow>
                              <mrow>
                                <mo>â</mo>
                              </mrow>
                            </msub>
                          </mrow>
                        </msub>
                      </mrow></semantics></math></span> concentration range for which the cells response distribution (phenotypes), during an induction experiment, is bimodal. That is, the larger the bimodal range, the less precise the switch is in order to generate a global response in the colony. In this regard, we point out that the precision of the switch in a noise-free situation is infinite since all cells achieve global coordination simultaneously.</p>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Figure <span class="InternalRef"><a href="#Fig4">4</a></span> shows, by means of a color density plot, the probability of a cell to have a particular GFP expression level after either 10 or 100 hours of induction as a function of <span class="InlineEquation" id="IEq37"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span>. In order to gather enough statistics, we average our results over 10 different realizations (i.e. experiments). For a large range of autoinducer concentrations, for both the <em xmlns="" class="EmphasisTypeItalic">lux01</em> and for the <em xmlns="" class="EmphasisTypeItalic">lux02</em> operon, the distribution of GFP expression after 10 h of induction is bimodal. As shown, some cells of the colony are induced before the critical concentration of the deterministic model at the steady state (black line). Still, the concentration for which more than 90% of the cells are induced requires up to four times more autoinducer than under deterministic conditions. Thus, on the one hand noise can help cells to get induced at lower autoinducer concentrations but, on the other hand, amplifies the non-stationary effects for achieving global coordination. In order to clarify this interplay between non-stationary and stochastic effects, we perform the same simulations with a larger induction time (100 h). As expected, the precision of the switch increases (10-fold change) and cells achieve global coordination at (<em xmlns="" class="EmphasisTypeItalic">lux01</em>) or before (<em xmlns="" class="EmphasisTypeItalic">lux02</em>) the critical deterministic concentration. Note that in all cases noise induces a significant variability in terms of the GFP expression levels in the high state compared to that of the low state (see also Figure <span class="InternalRef"><a href="#Fig5">5</a></span>). The variability introduced in the colony response by the fluctuations with respect to the deterministic approach can also be observed in experiments under weak inducing conditions where the autoinducer concentration is periodically decreased (see Additional file <span class="InternalRef"><a href="#MOESM4">4</a></span>: Figure S2).<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig4_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig4_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 4</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Cell response distribution to autoinducer induction in the stochastic model.</strong> Cell response probability after 10 hours (top: <strong class="EmphasisTypeBold">A</strong>, <strong class="EmphasisTypeBold">B</strong>) and 100 hours (middle: <strong class="EmphasisTypeBold">C</strong>, <strong class="EmphasisTypeBold">D</strong>) of induction at different autoinducer concentrations for the <em class="EmphasisTypeItalic">lux01</em> (left: <strong class="EmphasisTypeBold">A</strong>, <strong class="EmphasisTypeBold">C</strong>) and <em class="EmphasisTypeItalic">lux02</em> (right: <strong class="EmphasisTypeBold">B</strong>, <strong class="EmphasisTypeBold">D</strong>) operons in the stochastic model. The distribution reveals the coexistence of two subpopulations with low and high GFP expression when the cells are induced at intermediate autoinducer concentrations. The region of bistability (precision) is defined by the range of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq40"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> for which the response is bimodal according to the following criterion: the lower/upper limit of the bistable region (orange lines) is defined by the value of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq41"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> for which 90% of the cells are in the low/high state. The black line stands for the concentration of GFP (normalized) as a function of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq42"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> in the deterministic model at the steady state. After 10 hours of induction (top: <strong class="EmphasisTypeBold">A</strong>, <strong class="EmphasisTypeBold">B</strong>) most cells are still in a transient state if <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq43"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>&lt;</mo>
                                <mn>70</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span>. After 100 hours of induction (middle: <strong class="EmphasisTypeBold">C</strong>, <strong class="EmphasisTypeBold">D</strong>), the bimodality region shrinks and the precision increases. The population average curves of the induction and dilution experiments in the stochastic model (bottom: <strong class="EmphasisTypeBold">E</strong>, <strong class="EmphasisTypeBold">F</strong>, dashed lines) show that the intrinsic noise allows cells to jump to the high state inside the deterministic bistable region. On the other hand, the transition from high to low follows the deterministic path thus indicating that the switching rate in this case is close to zero.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig5_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig5_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 5</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Individual cell trajectories for autoinducer induction in the stochastic model.</strong> Individual cell trajectories (blue lines), cell population average (orange line) and deterministic solution (red dashed line) for an induction experiment at <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq44"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>=</mo>
                                <mn>25</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span> for the <em class="EmphasisTypeItalic">lux01</em> operon in the stochastic model. Individual cell trajectories show the heterogeneous distribution of cell jumping times. While some cells achieve full induction of the operon before the deterministic case, the global response of the population reaches steady-state at <span class="stix">â¼</span>30 hours, slower than the deterministic solution.</p>
                      </div></figcaption></figure></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">The heterogeneity in terms of the jumping statistics is revealed in Figure <span class="InternalRef"><a href="#Fig5">5</a></span> where we plot individual trajectories for the <em xmlns="" class="EmphasisTypeItalic">lux01</em> operon as a function of time at <span class="InlineEquation" id="IEq45"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>=</mo>
                        <mn>25</mn>
                        <mspace width="1em"></mspace>
                        <mtext mathvariant="italic">nM</mtext>
                      </mrow></semantics></math></span> over a period of 50 hours. Some cells become induced after 3 hours, while others need <span class="stix">â¼</span>10 times more induction time to reach the high state. At this concentration of autoinducer all cells have eventually reached the high state after <span class="stix">â¼</span>30 hours of induction. Importantly, we do not observe that cells jump back (see Discussion). That is, while there is variability over the colony in regards of the switching time, once the transition occurs the cell remains in the new state that is sustained over generations as seen in Figure <span class="InternalRef"><a href="#Fig6">6</a></span>. Therefore, over the typical timescale of an experiment (10 to 50 hours), the behaviour of the QS switch is highly dynamic and the precision of the switch is a transient quantity that crucially depends on the duration of induction.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig6_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig6_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 6</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Lineage tree of an induced population of cells in the stochastic model.</strong> Linage tree of a population of cells induced at a fixed autoinducer concentration <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq46"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                                <mo>=</mo>
                                <mn>50</mn>
                                <mtext>nM</mtext>
                              </mrow></semantics></math></span> for the <em class="EmphasisTypeItalic">lux01</em> operon (left) and the <em class="EmphasisTypeItalic">lux02</em> operon (right). Vertical lines represent individual cells and horizontal lines cell division events. The color of the lines is proportional to the normalized GFP expression. The initial number of cells is 100 and is kept constant during the experiment by âdeletingâ cells at random every time a cell divides (truncated vertical lines). The lineage tree shows how the state of the cell is transmitted over generations and reveals that once the operon is activated the transition is âirreversibleâ.</p>
                      </div></figcaption></figure></div>
                <p xmlns="http://www.w3.org/1999/xhtml" class="Para">As expected the intrinsic noise decreases the precision of the QS switch with respect to the deterministic case. Still, noise helps cells to become activated before the critical concentration of a fluctuations-free system under all induction conditions. Moreover, in steady-state conditions the high state is globally achieved before the critical deterministic concentration. This phenomenon is recapitulated in Figure <span class="InternalRef"><a href="#Fig4">4</a></span> (bottom) where we plot the population average response for the induction and dilution experiments at steady-state (100 h induction) for both the deterministic and stochastic models. Notice that the dilution curves of the stochastic model are similar to that of the deterministic model; however, the average transition to the high state occurs at a lower autoinducer concentration due to intrinsic fluctuations.</p>
              </section><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">The features of the QS switch depends on the transcriptional noise of LuxR</h3>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">For the same concentration of the external autoinducer, the stochastic dynamics of the regulatory network arises from the noise at the level of LuxI and LuxR. We now analyze the individual contribution of those by modulating the burst size of LuxR and LuxI, <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub>and <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>respectively. We notice that the burst size modulates the stochasticity levels while maintaining the average protein copy numbers. Additional file <span class="InternalRef"><a href="#MOESM5">5</a></span>: Figure S3 illustrates the effect of changing the burst size by showing individual trajectories of the chemical species obtained for large and small values of this quantity at low and high concentrations of the external autoinducer. In this regard, insight about the activation process can be obtained by computing the mean first passage time (MFPT) for transitions between the low and the high state. Figure <span class="InternalRef"><a href="#Fig7">7</a></span> shows this quantity as a function of <span class="InlineEquation" id="IEq47"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span> and for different values of the burst size of LuxR and LuxI. For the sake of comparison, we also compute the MFPT for the deterministic solution. We note that in that case, the MFPT inside the bistable region is infinite, since the deterministic system cannot spontaneously jump from one stable state to the other. Our results indicate that changing the burst size of LuxI does not modify the mean first passage time whereas changing the transcriptional noise at the level of LuxR modifies the jumping statistics. Moreover, our results reveal a non-trivial behaviour of the MFPT as a function of the concentration of the autoinducer. On one hand, with respect to the activation dynamics, when <span class="InlineEquation" id="IEq48"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span> is below <span class="stix">â¼</span>25 nM, an increase in LuxR noise decreases the mean time of the activation. That is, LuxR noise helps cells to get the initial activation quicker. On the other hand, above <span class="stix">â¼</span>25 nM of autoinducer concentration, the effect is the opposite: an increase in LuxR noise increases the mean jumping time thus slowing down the full cell activation.<figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig7_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig7_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 7</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Mean first passage time of cell activation for different burst size values.</strong> Mean first passage time of cell activation as a function of autoinducer concentration for different values of the burst size for LuxR (<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>) and LuxI (<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>) and for the deterministic solution: (<strong class="EmphasisTypeBold">A</strong>) low to high transition MFPT in the <em class="EmphasisTypeItalic">lux01</em> operon, (<strong class="EmphasisTypeBold">B</strong>) low to high transition MFPT in the <em class="EmphasisTypeItalic">lux02</em> operon. The lower (upper) limit of the shaded regions is the 10% (90%) quantile curve of the distribution of FPT for the cases <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=20 (blue shaded region) and <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=0.01 (green shaded region). The MFPT reveals a non-trivial behaviour: for low autoinducer concentration noise helps cells to jump quicker to the high state, while for high autoinducer concentration noise slows down the cells activation (see text). Intersections of the quantile 10% and quantile 90% curves with a horizontal line at <em class="EmphasisTypeItalic">t</em>=10h indicate the autoinducer concentration for which 10% of cell trajectories have jumped to the high state (left arrow) and the concentration for which 90% of cell trajectories have been activated (right arrow). The precision after 10h of induction (inversely proportional to the width of the region delimited by the arrows), increases when decreasing the noise in LuxR (see text). Note that in the case of the <em class="EmphasisTypeItalic">lux01</em> operon, we only change the value of <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>since GFP does not contribute to the activation process.</p>
                      </div></figcaption></figure></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">We observe these effects both for the <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> operons. Surprisingly, when the autoinducer concentration is above the critical concentration of the deterministic system, <span class="InlineEquation" id="IEq53"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                        <mo>â</mo>
                        <mn>20</mn>
                      </mrow></semantics></math></span> nM, the stochastic system always takes more time to get activated than the deterministic case. By computing additional properties of the first passage time probability density we also clarify the behaviour of the precision depending on the induction time. In particular, we compute the times <em xmlns="" class="EmphasisTypeItalic">t</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">low</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">t</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">high</em>
                  </sub> for which, at a given <span class="InlineEquation" id="IEq54"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mrow>
                            <mi>A</mi>
                          </mrow>
                          <mrow>
                            <mo>â</mo>
                          </mrow>
                        </msubsup>
                      </mrow></semantics></math></span> concentration, the probabilities of finding a FPT&lt;<em xmlns="" class="EmphasisTypeItalic">t</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">low</em>
                  </sub>and a FPT&gt;<em xmlns="" class="EmphasisTypeItalic">t</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">high</em>
                  </sub>are 10%, i.e. the 10% and 90% quantiles respectively. The shadings in Figure <span class="InternalRef"><a href="#Fig7">7</a></span> delimit these regions for the cases <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>=20 and <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub>=<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>=0.01. The precision of the switch after <em xmlns="" class="EmphasisTypeItalic">n</em> hours of induction, is directly related to the width of the shaded region at ãFPTã=<em xmlns="" class="EmphasisTypeItalic">n</em> h: at any given time, this width indicates which is the minimal concentration of autoinducer for getting 10% of cells already activated and also the concentration beyond which more than 90% of cells have been activated. Thus, in agreement with Figure <span class="InternalRef"><a href="#Fig4">4</a></span>, the induction time clearly modifies the precision: it increases (the width decreases) as the induction time becomes larger. Moreover, note that as the LuxR noise weakens the precision increases. Figure <span class="InternalRef"><a href="#Fig8">8</a></span> recapitulates some of these results. There we show the GFP expression probability for the <em xmlns="" class="EmphasisTypeItalic">lux02</em> operon after 10 hours of induction for different values of the burst size <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">R</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">I</em>
                  </sub>. Notice that the region of bimodality does not vary when changing the burst size for LuxI. However, decreasing the burst size in LuxR reduces the region of bimodality thus increasing the precision of the switch. Furthermore, the noise at the level of LuxR helps some cells to become activated at lower concentration levels of the autoinducer. Once more, this phenomenon does not depend on the levels of transcriptional noise of LuxI. That is, while the global coordination increases as the transcriptional noise of LuxR decreases, more concentration of the autoinducer is required to start activating cells. Figure <span class="InternalRef"><a href="#Fig7">7</a></span> also suggests that the sensitivity of the precision as a function of the induction time and/or as a function of the stochasticity levels get diminished after <span class="stix">â¼</span>30 hours since the width of the shaded region barely varies. Figure <span class="InternalRef"><a href="#Fig9">9</a></span> points towards that direction: under long induction time conditions (100 h) the precision of the switch remains constant regardless the value of the burst size. All together, these results indicate an interesting and counterintuitive role of the transcriptional noise of LuxR in terms of the biological function of the QS switch.<figure xmlns="" class="Figure" id="Fig8"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig8_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig8_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 8</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Cell response distribution in the transient regime for different burst size values.</strong> Cell response distribution (jumping probability) after 10 hours of induction (transient state) at different autoinducer concentrations for the <em class="EmphasisTypeItalic">lux02</em> operon in the stochastic model and different burst sizes. Burst size values (<strong class="EmphasisTypeBold">A</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=20 (<strong class="EmphasisTypeBold">B</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=4,b<sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=20 (<strong class="EmphasisTypeBold">C</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=20,b<sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=4 (<strong class="EmphasisTypeBold">D</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=4 (<strong class="EmphasisTypeBold">E</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=0.01. Width of bistable region: (<strong class="EmphasisTypeBold">A</strong>) = 60 nM (<strong class="EmphasisTypeBold">B</strong>) 25 nM (<strong class="EmphasisTypeBold">C</strong>) 70 nM (<strong class="EmphasisTypeBold">D</strong>) 27.5 nM (<strong class="EmphasisTypeBold">E</strong>) 25 nM. The black line stands for the concentration of GFP (normalized) as a function of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq55"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> in the deterministic model at the steady state.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig9"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig9_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig9_HTML.jpg">
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 9</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Cell response distribution at the steady-state for different burst size values.</strong> Cell response distribution at the steady-state (100 h induction), at different autoinducer concentrations for the <em class="EmphasisTypeItalic">lux02</em> operon in the stochastic model for different burst size values: (<strong class="EmphasisTypeBold">A</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=20 (<strong class="EmphasisTypeBold">B</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=4 (<strong class="EmphasisTypeBold">C</strong>) <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">R</em>
                          </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">I</em>
                          </sub>=0.01. The probability density of getting a particular GFP expression level is indicated by means of a density plot. The width of bistable region barely depends on the stochasticity levels, â7 nM. The black line stands for the concentration of GFP (normalized) as a function of <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq56"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                <msubsup>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>A</mi>
                                  </mrow>
                                  <mrow>
                                    <mo>â</mo>
                                  </mrow>
                                </msubsup>
                              </mrow></semantics></math></span> in the deterministic model at the steady state.</p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec13" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Discussion</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">The response of bacterial colonies driven by the QS signalling mechanism under noisy conditions has been addressed, in a broad sense, by different authors. In particular, the characterization of the collective response as a synchronization phenomenon where the phenotypic variations can be generically predicted has been proposed [<span class="CitationRef"><a href="#CR47">47</a></span>]. However, this approach requires gene regulatory interactions controlling the QS switch that do not induce bistability and lead to a monostable behaviour, e.g. negative feedback loops [<span class="CitationRef"><a href="#CR48">48</a></span>]. Our study focus on strains that display, as the wild-type LuxI/LuxR system do, bistability and, consequently, an alternative method to quantify the phenotypic variability induced by noise was needed, i.e. the precision concept. Moreover, previous works assume stationary conditions and disregard the role of the cell cycle duration. Herein, in agreement with experimental results, we have shown that the time for reaching a steady expression rate is much larger than the cell cycle duration (see [<span class="CitationRef"><a href="#CR10">10</a></span>]). As a result, we have revealed that the interplay between non-stationary and stochastic effects is key for understanding the global response of the colony and the phenotypic variability. Finally, we have shown that the intrinsic noise is able to stabilize a particular phenotypic state. This effect, namely the fluctuations inducing a slowing down in the activation of the cells, emerges because noise extends the bistable region compared to the deterministic system. While such a noise-induced phenomenon has been characterized in population models [<span class="CitationRef"><a href="#CR49">49</a></span>] and, more recently, in theoretical studies on bistable switches [<span class="CitationRef"><a href="#CR18">18</a></span>], to the best of our knowledge, this is the first time that is reported in the context of QS systems. All in all, from the viewpoint of the comprehension of how noisy inputs may condition phenotypic variability in bacterial colonies, our study introduces a number of advances.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Herein, we have characterized how the precision of the QS switch depends on the stochasticity levels and, importantly, elucidated which noisy component of the LuxI/LuxR regulatory network drives the observed phenomenology. Thus, we have found that under non-stationary conditions, LuxR controls the phenotypic variability and that changing the noise intensity at the level of LuxI has no effect on the precision of the switch. A plausible explanation for this reads as follows. The fluctuations at the level of LuxI are transmitted to the autoinducer. However, the diffusion mechanism rapidly averages out the stochasticity levels of the latter. This is not possible for LuxR which is kept within the cell. As a consequence the amount of activation complex, that is ultimately the responsible for the activation, is driven by the fluctuations of LuxR but not by those of LuxI.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Recent experimental work has measured the bioluminescence levels of individual <em xmlns="" class="EmphasisTypeItalic">V. fischeri</em> cells at fixed autoinducer concentration [<span class="CitationRef"><a href="#CR13">13</a></span>]. In agreement with our results, the authors observed that cells differed widely in terms of their activation time and luminescence distribution. Interestingly, other experiments have revealed the presence of additional regulatory interactions for controlling the LuxR noise levels. For example, C8HSL molecules, a second QS signal in <em xmlns="" class="EmphasisTypeItalic">V. fischeri</em>, has been suggested to reduce the noise in bioluminescence output of the cells at low autoinducer concentrations [<span class="CitationRef"><a href="#CR50">50</a></span>]. In the same direction, in <em xmlns="" class="EmphasisTypeItalic">V. harveyi</em>, the number of LuxR dimers is tightly regulated indicating a control over LuxR intrinsic noise [<span class="CitationRef"><a href="#CR51">51</a></span>]. In fact, wild-type <em xmlns="" class="EmphasisTypeItalic">V. harveyi</em> strains have two negative feedback loops that repress the production of LuxR [<span class="CitationRef"><a href="#CR52">52</a></span>] and this kind of regulatory circuit is known to reduce noise levels [<span class="CitationRef"><a href="#CR53">53</a></span>]. In this context, our results provide a feasible explanation for the network structure in wild-type strains: since noise in LuxR controls the phenotypic variability of the LuxR/LuxI QS systems, bacteria have evolved mechanisms to control its noise levels. An additional argument in this regard arises from our results about the deactivation of cells: once they are fully induced we do not observe reversibility of the phenotype (FPT larger than 100 h). First, these results are in agreement with other switching systems as the gallactose signalling network in yeast [<span class="CitationRef"><a href="#CR54">54</a></span>] and with theoretical results that explain the asymmetric switching dynamics due to stochastic effects [<span class="CitationRef"><a href="#CR18">18</a></span>]. Second, they reveal the importance of additional interactions that regulate negatively <em xmlns="" class="EmphasisTypeItalic">luxR</em> in wild-type strains and indicates that synthetic strains as <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> summarize many features of the wild-type operon during the activation process but fail to capture some of dynamical aspects of the deactivation phenomenon.</p><div xmlns="http://www.w3.org/1999/xhtml" class="Para">Finally, our simulations indicate that non-stationary effects are essential during the activation of the QS response. While speculative, these results can be extrapolated to growing colonies where the cell density is not kept constant. A good supply of nutrients implies short induction times since the concentration of autoinducer will quickly grow (exponentially) as the population size does. According to our results, this fast growing condition decreases the precision of the switch and, consequently, promotes variability at the population level (see Figure <span class="InternalRef"><a href="#Fig10">10</a></span>). In addition, the full collective activation of the system would require a large population size. On the other hand, if the colony grows in a poor nutrient environment, the system will have time to reach a steady-state more easily and the precision would increase. Hence, the variability would be diminished, and full activation would require smaller colony sizes. Most phenotypic changes induced by the QS mechanism refer to bacterial strategies for survival and/or colonization. In this context, our results suggest that both the QS activation threshold and the phenotypic variability might depend on the growth rate of the colony and, as a consequence, on the environmental conditions. This is in agreement with recent studies that show that the collective response of a population of cells depends not only on the underlying genetic circuit and the environmental signals, but also on the speed of variation of these signals [<span class="CitationRef"><a href="#CR55">55</a></span>].<figure xmlns="" class="Figure" id="Fig10"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                    <img src="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig10_HTML.jpg" alt="https://static-content.springer.com/image/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_Article_1066_Fig10_HTML.jpg">
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 10</span>
                      <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">The growth rate conditions the phenotypic variability.</strong> In the context of a growing colony, the autoinducer concentration increases as the colony does: purple lines show schematically two exponential growth conditions for the autoinducer concentration as a function of time. Our results on the MFPT, valid at fixed autoinducer concentrations, can be extrapolated, qualitatively, to the case of increasing autoinducer levels. Fast growth results in a large cell variability and large critical colony size for achieving a global response, while slow growth produces reduced cell variability and a smaller critical population size. Increasing fluctuations in LuxR have two opposite effects: in the slow growth case, increasing the noise (blue curves: <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">R</em>
                        </sub>=20; green curves: <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">R</em>
                        </sub>=0.01;) decreases the critical population size while hardly changing the variability, in the fast growth case, increasing noise increases the critical population size and increases greatly the variability.</p>
                    </div></figcaption></figure></div></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec14" class="Section1 RenderAsSection1"><h2 class="Heading js-ToggleCollapseSection">Conclusions</h2><div class="js-CollapseSection"><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Herein we have introduced deterministic and stochastic modelling approaches for describing the core functionality of the LuxI/LuxR regulatory network in quorum sensing systems. We have focused on synthetic constructs, <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em>, that reproduce the behaviour of the wild-type system and allow for controlled experiments that have provided quantification of the activation process [<span class="CitationRef"><a href="#CR10">10</a></span>]. The deterministic approach has allowed us to estimate different parameters of the model and reproduce the switch-like behaviour of the QS network. Thus, our simulations reveal that the interplay between non-stationary and stochastic effects are key and that, for an extended range of autoinducer concentrations, a bimodal phenotypic variability develops such that cells fail to produce a global response. In this context we have introduced the concept of precision of the QS switch, as the inverse of the width of the bimodal phenotypic region.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">By computing the statistics of the activation dynamics of cells, we have shown that the QS precision depends on the gene expression noise at the level of LuxR and is independent from that of LuxI. Our results, together with the experimental evidences on LuxR regulation in wild-type species, suggest that the noise at the level of LuxR controls the phenotypic variability of the LuxR/LuxI QS systems and that bacteria have evolved to control its intensity. In addition, the robust stabilization of the phenotype once is fully induced indicates that, albeit synthetic strains as <em xmlns="" class="EmphasisTypeItalic">lux01</em> and <em xmlns="" class="EmphasisTypeItalic">lux02</em> summarize many features of the wild-type operon during the activation process, they fail to capture crucial aspects of the deactivation phenomenon.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">Most insight in regards of the effect of LuxR noise on the dynamics of cell activation is given by the study of the mean first passage time (MFPT). In terms of the timing of activation, we have observed two opposite effects depending on the control parameter <span class="InlineEquation" id="IEq57"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                      <msubsup>
                        <mrow>
                          <mi>c</mi>
                        </mrow>
                        <mrow>
                          <mi>A</mi>
                        </mrow>
                        <mrow>
                          <mo>â</mo>
                        </mrow>
                      </msubsup>
                    </mrow></semantics></math></span>: for <span class="InlineEquation" id="IEq58"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                      <msubsup>
                        <mrow>
                          <mi>c</mi>
                        </mrow>
                        <mrow>
                          <mi>A</mi>
                        </mrow>
                        <mrow>
                          <mo>â</mo>
                        </mrow>
                      </msubsup>
                      <mo>â¾</mo>
                      <mn>20</mn>
                    </mrow></semantics></math></span> nM, the larger the noise in LuxR, the quicker the cells become activated, while for <span class="InlineEquation" id="IEq59"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                      <msubsup>
                        <mrow>
                          <mi>c</mi>
                        </mrow>
                        <mrow>
                          <mi>A</mi>
                        </mrow>
                        <mrow>
                          <mo>â</mo>
                        </mrow>
                      </msubsup>
                      <mo>â¿</mo>
                      <mn>20</mn>
                    </mrow></semantics></math></span> nM, we observe the opposite effect and noise slows down cell activation. We suggest that this effect can be explained by the stochastic stabilization of the low state. Moreover, the calculation of additional properties of the statistics of the first passage time have allowed us to relate the concept of precision of the switch with the variability of the FPT by estimating the 10% and 90% quantiles.</p><p xmlns="http://www.w3.org/1999/xhtml" class="Para">In summary, our results indicate that in bacterial colonies driven by the QS mechanism there is a trade-off between the activation onset and a global response due to non-stationary and stochastic effects. On one hand, large levels of noise at the level of LuxR imply that cells require smaller autoinducer levels for achieving an activation onset but, at the same time, a global response requires a substantial autoinducer concentration. On the other hand, if the LuxR noise levels are small, the activation onset is shifted toward larger values of the autoinducer concentration but the global response is achieved for smaller concentration values. Our study could be useful for Synthetic Biology approaches that exploit the QS mechanism. The fact that some important features of the QS mechanism, e.g. precision, rely on the burst size of one component, opens the door to modifications of the LuxI/LuxR operon for regulating the response depending on the problem under consideration. Finally, further research is needed about the general validity and applicability on the noise-induced stabilization phenomenon of particular phenotypic states in other gene regulatory systems beyond the QS mechanism. Work in that direction is in progress.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1" id="Declarations"><h2 class="Heading js-ToggleCollapseSection">
              Declarations
            </h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We thank Oriol Canela Xandri and Nico Geisel for fruitful comments. Financial support was provided by MICINN under grant BFU2010-21847-C02-01/BMC, and by DURSI through project 2009-SGR/01055. We also acknowledge support from the European Science Foundation through the FuncDyn programme. M.W. acknowledges the support of the Spanish MICINN through a doctoral fellowship (FPU AP2008-03272).</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Electronic supplementary material</h3><div id="widgetContainer"></div>
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM1_ESM.pdf">12918_2012_1066_MOESM1_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Additional file 1:Text S1. Chemical equations for the deterministic model. (PDF 107 KB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item type-video"><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM2_ESM.avi">12918_2012_1066_MOESM2_ESM.avi</a>
                          <span xmlns="" class="SimplePara">Additional file 2:<strong class="EmphasisTypeBold">Video S1.</strong> Movie of the stochastic simulation. Movie of the stochastic simulation for the <em class="EmphasisTypeItalic">lux02</em> operon, 10 h of induction at <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq1"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>*</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>50</mn>
                                  <mtext mathvariant="italic">nM</mtext>
                                </mrow></semantics></math></span>, burst size <em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                              <em xmlns="" class="EmphasisTypeItalic">R</em>
                            </sub>=<em class="EmphasisTypeItalic">b</em><sub xmlns="http://www.w3.org/1999/xhtml">
                              <em xmlns="" class="EmphasisTypeItalic">I</em>
                            </sub>=4. Cells are modelled as individual compartments containing a copy of the LuxR/LuxI regulatory network. The Gillespie algorithm (see text for details) is used to integrate the stochastic dynamics of the whole system of cells. Cell growth and division is explicitly taken into account as well as a certain degree of stochasticity in the cell cycle duration. Cells movement is purely aesthetic since we do not include any spatial effects in our model and consider a well-mixed environment. The number of cells (<em class="EmphasisTypeItalic">N</em>=100) is maintained constant by removing one cell at random each time a cell divides. (AVI 19 MB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM3_ESM.png">12918_2012_1066_MOESM3_ESM.png</a>
                          <span xmlns="" class="SimplePara">Additional file 3:<strong class="EmphasisTypeBold">Figure S1.</strong> Intra and extracellular autoinducer as a function of exogeneous autoinducer concentration. Response curves to autoinducer induction for <em class="EmphasisTypeItalic">lux01</em> (<strong class="EmphasisTypeBold">A</strong>, <strong class="EmphasisTypeBold">C</strong> and <strong class="EmphasisTypeBold">E</strong>) and <em class="EmphasisTypeItalic">lux02</em> (<strong class="EmphasisTypeBold">B</strong>, <strong class="EmphasisTypeBold">D</strong> and <strong class="EmphasisTypeBold">E</strong>) operons. Total autoinducer concentration <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq10"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msub>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>A</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">tot</mtext>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                </mrow></semantics></math></span> in the external volume and in the cells (<strong class="EmphasisTypeBold">A</strong> and <strong class="EmphasisTypeBold">B</strong>), intracellular concentration <em class="EmphasisTypeItalic">c</em><sub xmlns="http://www.w3.org/1999/xhtml">
                              <em xmlns="" class="EmphasisTypeItalic">A</em>
                            </sub>(<strong class="EmphasisTypeBold">C</strong> and <strong class="EmphasisTypeBold">D</strong>), and extracellular concentration <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq11"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msub>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>A</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">ext</mtext>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                </mrow></semantics></math></span> (<strong class="EmphasisTypeBold">E</strong> and <strong class="EmphasisTypeBold">F</strong>), as a function of the exogenous autoinducer concentration, <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq12"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                </mrow></semantics></math></span>, in the deterministic model. All graphs represent the steady-state response for increasing (blue curve) and decreasing (red curve) autoinducer concentrations. The exogeneous autoinducer concentration <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq13"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                </mrow></semantics></math></span> controls the autoinducer concentration in the medium by means of an influx and an efflux (see main text). Upon activation of the operon, LuxR is produced at high levels, thus sequestering autoinducer molecules inside the cells. The bound form of autoinducer cannot diffuse out of the cell and is therefore not subjected to the influx and efflux. This explains why the total concentration of autoinducer in the system, <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq14"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msub>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>A</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">tot</mtext>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <mfrac>
                                    <mrow>
                                      <mn>1</mn>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>V</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">tot</mtext>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </mfrac>
                                  <mfenced separators="" open="[" close="]">
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>V</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">cell</mtext>
                                        </mrow>
                                      </msub>
                                      <mfenced separators="" open="(" close=")">
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>c</mi>
                                            </mrow>
                                            <mrow>
                                              <mi>A</mi>
                                            </mrow>
                                          </msub>
                                          <mo>+</mo>
                                          <msub>
                                            <mrow>
                                              <mi>c</mi>
                                            </mrow>
                                            <mrow>
                                              <mtext mathvariant="italic">luxR</mtext>
                                              <mo>Â·</mo>
                                              <mi>A</mi>
                                            </mrow>
                                          </msub>
                                          <mo>+</mo>
                                          <msub>
                                            <mrow>
                                              <mi>c</mi>
                                            </mrow>
                                            <mrow>
                                              <msub>
                                                <mrow>
                                                  <mfenced separators="" open="(" close=")">
                                                    <mrow>
                                                      <mtext mathvariant="italic">luxR</mtext>
                                                      <mo>Â·</mo>
                                                      <mi>A</mi>
                                                    </mrow>
                                                  </mfenced>
                                                </mrow>
                                                <mrow>
                                                  <mn>2</mn>
                                                </mrow>
                                              </msub>
                                            </mrow>
                                          </msub>
                                          <mo>+</mo>
                                          <msub>
                                            <mrow>
                                              <mi>c</mi>
                                            </mrow>
                                            <mrow>
                                              <mtext mathvariant="italic">DNA</mtext>
                                              <mo>Â·</mo>
                                              <msub>
                                                <mrow>
                                                  <mfenced separators="" open="(" close=")">
                                                    <mrow>
                                                      <mtext mathvariant="italic">luxR</mtext>
                                                      <mo>Â·</mo>
                                                      <mi>A</mi>
                                                    </mrow>
                                                  </mfenced>
                                                </mrow>
                                                <mrow>
                                                  <mn>2</mn>
                                                </mrow>
                                              </msub>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </mfenced>
                                      <mo>+</mo>
                                      <msub>
                                        <mrow>
                                          <mi>V</mi>
                                        </mrow>
                                        <mrow>
                                          <mtext mathvariant="italic">ext</mtext>
                                        </mrow>
                                      </msub>
                                      <msub>
                                        <mrow>
                                          <mi>c</mi>
                                        </mrow>
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>A</mi>
                                            </mrow>
                                            <mrow>
                                              <mtext mathvariant="italic">ext</mtext>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </mfenced>
                                </mrow></semantics></math></span> is slightly larger than <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq15"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                </mrow></semantics></math></span>, when the operon is activated. For the same reason, the free form of autoinducer, both in the cell and in the medium, is slightly smaller. (PNG 261 KB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM4_ESM.png">12918_2012_1066_MOESM4_ESM.png</a>
                          <span xmlns="" class="SimplePara">Additional file 4:<strong class="EmphasisTypeBold">Figure S2.</strong> Cell response distribution during decreasing-concentration trajectories. Cell response distribution for decreasing-concentration trajectories for <em class="EmphasisTypeItalic">lux01</em> (left) and <em class="EmphasisTypeItalic">lux02</em> (right) strains in the stochastic model. Cells are initially induced at <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq38"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>100</mn>
                                  <mtext>nM</mtext>
                                </mrow></semantics></math></span> for 2 hours. The concentration of exogenous autoinducer <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq39"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                </mrow></semantics></math></span> is then hourly decreased in order to simulate the experiments (see [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR10">10</a></span>]). The cell distribution reveals the variety of cell trajectories in comparison to the deterministic population average solution (green line). The cells jump to the high state for a wide range of times and autoinducer concentrations. Note also that fluctuations leads to a stabilization of the low state with respect to the deterministic solution. (PNG 511 KB)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM5_ESM.png">12918_2012_1066_MOESM5_ESM.png</a>
                          <span xmlns="" class="SimplePara">Additional file 5:<strong class="EmphasisTypeBold">Figure S3.</strong> Trajectory of chemical species in individual cells. Trajectory of chemical species LuxR mRNA (mR), LuxR, LuxI, intracellular autoinducer (AI), regulatory complex (<em class="EmphasisTypeItalic">LuxR</em>Â·<em class="EmphasisTypeItalic">AI</em>)<sub xmlns="http://www.w3.org/1999/xhtml">2</sub>(AL2) and promoter bound to complex (P10), in an individual cell for the following control parameter and burst size values: (A) <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq49"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>15</mn>
                                  <mspace width="0.3em"></mspace>
                                  <mtext>nM</mtext>
                                  <mo>,</mo>
                                  <mspace width="1em"></mspace>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <mn>20</mn>
                                </mrow></semantics></math></span>, (B) <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq50"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>50</mn>
                                  <mspace width="0.3em"></mspace>
                                  <mtext>nM</mtext>
                                  <mo>,</mo>
                                  <mspace width="1em"></mspace>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <mn>20</mn>
                                </mrow></semantics></math></span>, (C) <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq51"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>15</mn>
                                  <mspace width="0.3em"></mspace>
                                  <mtext>nM</mtext>
                                  <mo>,</mo>
                                  <mspace width="1em"></mspace>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <mn>0</mn>
                                  <mi>.</mi>
                                  <mn>01</mn>
                                </mrow></semantics></math></span>, (D) <span xmlns="http://www.w3.org/1999/xhtml" class="InlineEquation" id="IEq52"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                                  <msubsup>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <mo>â</mo>
                                    </mrow>
                                  </msubsup>
                                  <mo>=</mo>
                                  <mn>50</mn>
                                  <mspace width="0.3em"></mspace>
                                  <mtext>nM</mtext>
                                  <mo>,</mo>
                                  <mspace width="1em"></mspace>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>I</mi>
                                    </mrow>
                                  </msub>
                                  <mo>=</mo>
                                  <mn>0</mn>
                                  <mi>.</mi>
                                  <mn>01</mn>
                                </mrow></semantics></math></span>. (PNG 381 KB)</span>
                        </div></div>
                  </div>
                </div>
              
              
                
                <div xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authorsâ original submitted files for images.<div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM6_ESM.tiff">12918_2012_1066_MOESM6_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM7_ESM.tiff">12918_2012_1066_MOESM7_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 2</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM8_ESM.tiff">12918_2012_1066_MOESM8_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 3</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM9">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM9_ESM.tiff">12918_2012_1066_MOESM9_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 4</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM10">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM10_ESM.tiff">12918_2012_1066_MOESM10_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 5</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM11">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM11_ESM.tiff">12918_2012_1066_MOESM11_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 6</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM12">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM12_ESM.tiff">12918_2012_1066_MOESM12_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 7</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM13">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM13_ESM.tiff">12918_2012_1066_MOESM13_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 8</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM14">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM14_ESM.tiff">12918_2012_1066_MOESM14_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 9</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM15">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1752-0509-7-6/MediaObjects/12918_2012_1066_MOESM15_ESM.tiff">12918_2012_1066_MOESM15_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authorsâ original file for figure 10</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authorsâ contributions</h3><p xmlns="" class="SimplePara">MW and JB designed the experiments. MW carried out the simulations. MW and JB analyzed the data. All authors read and approved the final manuscript.</p></aside></div></section></div>
                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection">Authorsâ Affiliations</h2><div class="js-CollapseSection"><div class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Computer Simulation and Modelling (Co.S.Mo.) Lab, Parc CientÃ­fic de Barcelona</div></div></div></section></div>
                            <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection">References</h2><div class="js-CollapseSection"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1">Bassler BL, Losick R: Bacterially speaking. Cell. 2006, 125 (2): 237-46. 10.1016/j.cell.2006.04.001.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16630813"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.cell.2006.04.001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Bacterially%20speaking&amp;author=BL.%20Bassler&amp;author=R.%20Losick&amp;journal=Cell&amp;volume=125&amp;issue=2&amp;pages=237-46&amp;publication_year=2006&amp;doi=10.1016%2Fj.cell.2006.04.001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2">Saka Y, Lhoussaine C, Kuttler C, Ullner E, Thiel M: Theoretical basis of the community effect in development. BMC Syst Biol. 2011, 5: 54-10.1186/1752-0509-5-54.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21496342"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3105943"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1186/1752-0509-5-54"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Theoretical%20basis%20of%20the%20community%20effect%20in%20development&amp;author=Y.%20Saka&amp;author=C.%20Lhoussaine&amp;author=C.%20Kuttler&amp;author=E.%20Ullner&amp;author=M.%20Thiel&amp;journal=BMC%20Syst%20Biol&amp;volume=5&amp;pages=54&amp;publication_year=2011&amp;doi=10.1186%2F1752-0509-5-54"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3">Antunes LCM, Ferreira RBR, Buckner MMC, Finlay BB: Quorum sensing in bacterial virulence. Microbiol (Reading, England). 2010, 156 (Pt 8): 2271-82.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1099/mic.0.038794-0"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Quorum%20sensing%20in%20bacterial%20virulence&amp;author=LCM.%20Antunes&amp;author=RBR.%20Ferreira&amp;author=MMC.%20Buckner&amp;author=BB.%20Finlay&amp;journal=Microbiol%20%28Reading%2C%20England%29&amp;volume=156&amp;issue=Pt%208&amp;pages=2271-82&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4">Pai A, Tanouchi Y, Collins CH, You L: Engineering multicellular systems by cell-cell communication. Curr Opin Biotechnol. 2009, 20 (4): 461-70. 10.1016/j.copbio.2009.08.006.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19733047"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2763981"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.copbio.2009.08.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Engineering%20multicellular%20systems%20by%20cell-cell%20communication&amp;author=A.%20Pai&amp;author=Y.%20Tanouchi&amp;author=CH.%20Collins&amp;author=L.%20You&amp;journal=Curr%20Opin%20Biotechnol&amp;volume=20&amp;issue=4&amp;pages=461-70&amp;publication_year=2009&amp;doi=10.1016%2Fj.copbio.2009.08.006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5">Anderson JC, Clarke EJ, Arkin AP: Voigt Ca: Environmentally controlled invasion of cancer cells by engineered bacteria. J Mol Biol. 2006, 355 (4): 619-27. 10.1016/j.jmb.2005.10.076.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16330045"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.jmb.2005.10.076"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Voigt%20Ca%3A%20Environmentally%20controlled%20invasion%20of%20cancer%20cells%20by%20engineered%20bacteria&amp;author=JC.%20Anderson&amp;author=EJ.%20Clarke&amp;author=AP.%20Arkin&amp;journal=J%20Mol%20Biol&amp;volume=355&amp;issue=4&amp;pages=619-27&amp;publication_year=2006&amp;doi=10.1016%2Fj.jmb.2005.10.076"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6">Ng WL, Bassler BL: Bacterial quorum-sensing network architectures. Annu Rev Genet. 2009, 43: 197-222. 10.1146/annurev-genet-102108-134304.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19686078"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4313539"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1146/annurev-genet-102108-134304"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Bacterial%20quorum-sensing%20network%20architectures&amp;author=WL.%20Ng&amp;author=BL.%20Bassler&amp;journal=Annu%20Rev%20Genet&amp;volume=43&amp;pages=197-222&amp;publication_year=2009&amp;doi=10.1146%2Fannurev-genet-102108-134304"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7">Dockery JD, Keener JP: A mathematical model for quorum sensing in Pseudomonas aeruginosa. Bull Math Biol. 2001, 63: 95-116. 10.1006/bulm.2000.0205.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11146885"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1006/bulm.2000.0205"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=A%20mathematical%20model%20for%20quorum%20sensing%20in%20Pseudomonas%20aeruginosa&amp;author=JD.%20Dockery&amp;author=JP.%20Keener&amp;journal=Bull%20Math%20Biol&amp;volume=63&amp;pages=95-116&amp;publication_year=2001&amp;doi=10.1006%2Fbulm.2000.0205"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8">Cox C, Peterson G, Allen M, Lancaster J, McCollum J, Austin D, Yan L, Sayler G, Simpson M: Analysis of noise in quorum sensing. OMICS. 2003, 7 (3): 317-334. 10.1089/153623103322452422.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14583119"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1089/153623103322452422"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Analysis%20of%20noise%20in%20quorum%20sensing&amp;author=C.%20Cox&amp;author=G.%20Peterson&amp;author=M.%20Allen&amp;author=J.%20Lancaster&amp;author=J.%20McCollum&amp;author=D.%20Austin&amp;author=L.%20Yan&amp;author=G.%20Sayler&amp;author=M.%20Simpson&amp;journal=OMICS&amp;volume=7&amp;issue=3&amp;pages=317-334&amp;publication_year=2003&amp;doi=10.1089%2F153623103322452422"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9">Goryachev AB, Toh DJ, Lee T: Systems analysis of a quorum sensing network: design constraints imposed by the functional requirements, network topology and kinetic constants. Bio Syst. 2006, 83 (2-3): 178-87.<span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Systems%20analysis%20of%20a%20quorum%20sensing%20network%3A%20design%20constraints%20imposed%20by%20the%20functional%20requirements%2C%20network%20topology%20and%20kinetic%20constants&amp;author=AB.%20Goryachev&amp;author=DJ.%20Toh&amp;author=T.%20Lee&amp;journal=Bio%20Syst&amp;volume=83&amp;issue=2-3&amp;pages=178-87&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10">Williams JW, Cui X, Levchenko A, Stevens AM: Robust and sensitive control of a quorum-sensing circuit by two interlocked feedback loops. Mol Syst Biol. 2008, 4 (234): 234-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19096361"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2615304"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Robust%20and%20sensitive%20control%20of%20a%20quorum-sensing%20circuit%20by%20two%20interlocked%20feedback%20loops&amp;author=JW.%20Williams&amp;author=X.%20Cui&amp;author=A.%20Levchenko&amp;author=AM.%20Stevens&amp;journal=Mol%20Syst%20Biol&amp;volume=4&amp;issue=234&amp;pages=234&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11">Haseltine EL, Arnold FH: Implications of rewiring bacterial quorum sensing. Appl Environ Microbiol. 2008, 74 (2): 437-45. 10.1128/AEM.01688-07.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18039819"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2223271"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1128/AEM.01688-07"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Implications%20of%20rewiring%20bacterial%20quorum%20sensing&amp;author=EL.%20Haseltine&amp;author=FH.%20Arnold&amp;journal=Appl%20Environ%20Microbiol&amp;volume=74&amp;issue=2&amp;pages=437-45&amp;publication_year=2008&amp;doi=10.1128%2FAEM.01688-07"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12">Anetzberger C, Pirch T, Jung K: Heterogeneity in quorum sensing-regulated bioluminescence of Vibrio harveyi. Mol Microbiol. 2009, 73 (2): 267-77. 10.1111/j.1365-2958.2009.06768.x.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19555459"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1111/j.1365-2958.2009.06768.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Heterogeneity%20in%20quorum%20sensing-regulated%20bioluminescence%20of%20Vibrio%20harveyi&amp;author=C.%20Anetzberger&amp;author=T.%20Pirch&amp;author=K.%20Jung&amp;journal=Mol%20Microbiol&amp;volume=73&amp;issue=2&amp;pages=267-77&amp;publication_year=2009&amp;doi=10.1111%2Fj.1365-2958.2009.06768.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13">PÃ©rez PD, Hagen SJ: Heterogeneous response to a quorum-sensing signal in the luminescence of individual Vibrio fischeri. PLoS One. 2010, 5 (11): e15473-10.1371/journal.pone.0015473.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21103327"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2982848"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0015473"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Heterogeneous%20response%20to%20a%20quorum-sensing%20signal%20in%20the%20luminescence%20of%20individual%20Vibrio%20fischeri&amp;author=PD.%20P%C3%A9rez&amp;author=SJ.%20Hagen&amp;journal=PLoS%20One&amp;volume=5&amp;issue=11&amp;pages=e15473&amp;publication_year=2010&amp;doi=10.1371%2Fjournal.pone.0015473"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14">Boedicker JQ, Vincent ME, Ismagilov RF: Microfluidic confinement of single cells of bacteria in small volumes initiates high-density behavior of quorum sensing and growth and reveals its variability. Angew Chem Int Ed Engl. 2009, 48 (32): 5908-11. 10.1002/anie.200901550.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19565587"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2748941"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1002/anie.200901550"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Microfluidic%20confinement%20of%20single%20cells%20of%20bacteria%20in%20small%20volumes%20initiates%20high-density%20behavior%20of%20quorum%20sensing%20and%20growth%20and%20reveals%20its%20variability&amp;author=JQ.%20Boedicker&amp;author=ME.%20Vincent&amp;author=RF.%20Ismagilov&amp;journal=Angew%20Chem%20Int%20Ed%20Engl&amp;volume=48&amp;issue=32&amp;pages=5908-11&amp;publication_year=2009&amp;doi=10.1002%2Fanie.200901550"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15">Hagen SJ, Son M, Weiss JT, Young JH: Bacterium in a box: sensing of quorum and environment by the LuxI/LuxR gene regulatory circuit. J Biol Phys. 2010, 36 (3): 317-327. 10.1007/s10867-010-9186-4.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21629592"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2868974"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1007/s10867-010-9186-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Bacterium%20in%20a%20box%3A%20sensing%20of%20quorum%20and%20environment%20by%20the%20LuxI%2FLuxR%20gene%20regulatory%20circuit&amp;author=SJ.%20Hagen&amp;author=M.%20Son&amp;author=JT.%20Weiss&amp;author=JH.%20Young&amp;journal=J%20Biol%20Phys&amp;volume=36&amp;issue=3&amp;pages=317-327&amp;publication_year=2010&amp;doi=10.1007%2Fs10867-010-9186-4"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16">Tian T, Burrage K: Stochastic models for regulatory networks of the genetic toggle switch. Proc Natl Acad Sci U S A. 2006, 103 (22): 8372-7. 10.1073/pnas.0507818103.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16714385"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1482501"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1073/pnas.0507818103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Stochastic%20models%20for%20regulatory%20networks%20of%20the%20genetic%20toggle%20switch&amp;author=T.%20Tian&amp;author=K.%20Burrage&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=103&amp;issue=22&amp;pages=8372-7&amp;publication_year=2006&amp;doi=10.1073%2Fpnas.0507818103"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17">Wang J, Zhang J, Yuan Z, Zhou T: Noise-induced switches in network systems of the genetic toggle switch. BMC Syst Biol. 2007, 1: 50-10.1186/1752-0509-1-50.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18005421"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2214838"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1186/1752-0509-1-50"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Noise-induced%20switches%20in%20network%20systems%20of%20the%20genetic%20toggle%20switch&amp;author=J.%20Wang&amp;author=J.%20Zhang&amp;author=Z.%20Yuan&amp;author=T.%20Zhou&amp;journal=BMC%20Syst%20Biol&amp;volume=1&amp;pages=50&amp;publication_year=2007&amp;doi=10.1186%2F1752-0509-1-50"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18">Frigola D, Casanellas L, Sancho JM, IbaÃ±es M: Asymmetric stochastic switching driven by intrinsic molecular noise. PLoS One. 2012, 7 (2): e31407-10.1371/journal.pone.0031407.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22363638"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3283640"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0031407"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Asymmetric%20stochastic%20switching%20driven%20by%20intrinsic%20molecular%20noise&amp;author=D.%20Frigola&amp;author=L.%20Casanellas&amp;author=JM.%20Sancho&amp;author=M.%20Iba%C3%B1es&amp;journal=PLoS%20One&amp;volume=7&amp;issue=2&amp;pages=e31407&amp;publication_year=2012&amp;doi=10.1371%2Fjournal.pone.0031407"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19">Danino T, MondragÃ³n-Palomino O, Tsimring L, Hasty J: A synchronized quorum of genetic clocks. Nature. 2010, 463 (7279): 326-330. 10.1038/nature08753.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20090747"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838179"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nature08753"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=A%20synchronized%20quorum%20of%20genetic%20clocks&amp;author=T.%20Danino&amp;author=O.%20Mondrag%C3%B3n-Palomino&amp;author=L.%20Tsimring&amp;author=J.%20Hasty&amp;journal=Nature&amp;volume=463&amp;issue=7279&amp;pages=326-330&amp;publication_year=2010&amp;doi=10.1038%2Fnature08753"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20">GarcÃ­a-Ojalvo J, Elowitz M, Strogatz S: Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing. Proc Natl Acad Sci U S A. 2004, 101 (30): 10955-10.1073/pnas.0307095101.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15256602"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC503725"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1073/pnas.0307095101"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Modeling%20a%20synthetic%20multicellular%20clock%3A%20Repressilators%20coupled%20by%20quorum%20sensing&amp;author=J.%20Garc%C3%ADa-Ojalvo&amp;author=M.%20Elowitz&amp;author=S.%20Strogatz&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=101&amp;issue=30&amp;pages=10955&amp;publication_year=2004&amp;doi=10.1073%2Fpnas.0307095101"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21">Karlsson D, Karlsson S, Gustafsson E, Normark BH, Nilsson P: Modeling the regulation of the competence-evoking quorum sensing network in Streptococcus pneumoniae. BioSystems. 2007, 90: 211-23. 10.1016/j.biosystems.2006.08.005.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17011700"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.biosystems.2006.08.005"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Modeling%20the%20regulation%20of%20the%20competence-evoking%20quorum%20sensing%20network%20in%20Streptococcus%20pneumoniae&amp;author=D.%20Karlsson&amp;author=S.%20Karlsson&amp;author=E.%20Gustafsson&amp;author=BH.%20Normark&amp;author=P.%20Nilsson&amp;journal=BioSystems&amp;volume=90&amp;pages=211-23&amp;publication_year=2007&amp;doi=10.1016%2Fj.biosystems.2006.08.005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22">Kuttler C, Hense BA: Interplay of two quorum sensing regulation systems of Vibrio fischeri. J Theor Biol. 2008, 251: 167-80. 10.1016/j.jtbi.2007.11.015.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18164038"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.jtbi.2007.11.015"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Interplay%20of%20two%20quorum%20sensing%20regulation%20systems%20of%20Vibrio%20fischeri&amp;author=C.%20Kuttler&amp;author=BA.%20Hense&amp;journal=J%20Theor%20Biol&amp;volume=251&amp;pages=167-80&amp;publication_year=2008&amp;doi=10.1016%2Fj.jtbi.2007.11.015"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23">Tanouchi Y, Tu D, Kim J, You L: Noise reduction by diffusional dissipation in a minimal quorum sensing motif. PLoS Comput Biol. 2008, 4 (8): e1000167-10.1371/journal.pcbi.1000167.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18769706"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2507755"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pcbi.1000167"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Noise%20reduction%20by%20diffusional%20dissipation%20in%20a%20minimal%20quorum%20sensing%20motif&amp;author=Y.%20Tanouchi&amp;author=D.%20Tu&amp;author=J.%20Kim&amp;author=L.%20You&amp;journal=PLoS%20Comput%20Biol&amp;volume=4&amp;issue=8&amp;pages=e1000167&amp;publication_year=2008&amp;doi=10.1371%2Fjournal.pcbi.1000167"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24">Pai A, You L: Optimal tuning of bacterial sensing potential. Mol Syst Biol. 2009, 5 (286): 286-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19584835"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2724973"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Optimal%20tuning%20of%20bacterial%20sensing%20potential&amp;author=A.%20Pai&amp;author=L.%20You&amp;journal=Mol%20Syst%20Biol&amp;volume=5&amp;issue=286&amp;pages=286&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25">Goryachev AB, Toh DJ, Wee KB, Zhang HB, Zhang LH, Lee T: Transition to quorum sensing in an Agrobacterium population: A stochastic model. PLoS Comput Biol. 2005, 1 (4): e37-10.1371/journal.pcbi.0010037.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16170413"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1214540"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pcbi.0010037"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Transition%20to%20quorum%20sensing%20in%20an%20Agrobacterium%20population%3A%20A%20stochastic%20model&amp;author=AB.%20Goryachev&amp;author=DJ.%20Toh&amp;author=KB.%20Wee&amp;author=HB.%20Zhang&amp;author=LH.%20Zhang&amp;author=T.%20Lee&amp;journal=PLoS%20Comput%20Biol&amp;volume=1&amp;issue=4&amp;pages=e37&amp;publication_year=2005&amp;doi=10.1371%2Fjournal.pcbi.0010037"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26">Romero-Campero FJ, PÃ©rez-JimÃ©nez MJ: A model of the quorum sensing system in Vibrio fischeri using P systems. Artif Life. 2008, 14: 95-109. 10.1162/artl.2008.14.1.95.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18171133"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1162/artl.2008.14.1.95"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=A%20model%20of%20the%20quorum%20sensing%20system%20in%20Vibrio%20fischeri%20using%20P%20systems&amp;author=FJ.%20Romero-Campero&amp;author=MJ.%20P%C3%A9rez-Jim%C3%A9nez&amp;journal=Artif%20Life&amp;volume=14&amp;pages=95-109&amp;publication_year=2008&amp;doi=10.1162%2Fartl.2008.14.1.95"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27">Koseska A, Zaikin A, Kurths J, GarcÃ­a-Ojalvo J: Timing cellular decision making under noise via cell-cell communication. PLoS One. 2009, 4 (3): e4872-10.1371/journal.pone.0004872.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19283068"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2652718"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0004872"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Timing%20cellular%20decision%20making%20under%20noise%20via%20cell-cell%20communication&amp;author=A.%20Koseska&amp;author=A.%20Zaikin&amp;author=J.%20Kurths&amp;author=J.%20Garc%C3%ADa-Ojalvo&amp;journal=PLoS%20One&amp;volume=4&amp;issue=3&amp;pages=e4872&amp;publication_year=2009&amp;doi=10.1371%2Fjournal.pone.0004872"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28">Melke P, Sahlin P, Levchenko A, JÃ¶nsson H: A cell-based model for quorum sensing in heterogeneous bacterial colonies. PLoS Comput Biol. 2010, 6 (6): e1000819-10.1371/journal.pcbi.1000819.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20585545"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2887461"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pcbi.1000819"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=A%20cell-based%20model%20for%20quorum%20sensing%20in%20heterogeneous%20bacterial%20colonies&amp;author=P.%20Melke&amp;author=P.%20Sahlin&amp;author=A.%20Levchenko&amp;author=H.%20J%C3%B6nsson&amp;journal=PLoS%20Comput%20Biol&amp;volume=6&amp;issue=6&amp;pages=e1000819&amp;publication_year=2010&amp;doi=10.1371%2Fjournal.pcbi.1000819"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29">Lyell NL, Dunn AK, Bose JL, Stabb EV: Bright mutants of Vibrio fischeri ES114 reveal conditions and regulators that control bioluminescence and expression of the lux operon. J Bacteriol. 2010, 192 (19): 5103-14. 10.1128/JB.00524-10.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20693328"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2944500"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1128/JB.00524-10"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Bright%20mutants%20of%20Vibrio%20fischeri%20ES114%20reveal%20conditions%20and%20regulators%20that%20control%20bioluminescence%20and%20expression%20of%20the%20lux%20operon&amp;author=NL.%20Lyell&amp;author=AK.%20Dunn&amp;author=JL.%20Bose&amp;author=EV.%20Stabb&amp;journal=J%20Bacteriol&amp;volume=192&amp;issue=19&amp;pages=5103-14&amp;publication_year=2010&amp;doi=10.1128%2FJB.00524-10"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30">Sitnikov D, Shadel G, Baldwin T: Autoinducer-independent mutants of the LuxR transcriptional activator exhibit differential effects on the twolux promoters ofVibrio fischeri. Mol Syst Biol. 1996, 252 (5): 622-625.<span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Autoinducer-independent%20mutants%20of%20the%20LuxR%20transcriptional%20activator%20exhibit%20differential%20effects%20on%20the%20twolux%20promoters%20ofVibrio%20fischeri&amp;author=D.%20Sitnikov&amp;author=G.%20Shadel&amp;author=T.%20Baldwin&amp;journal=Mol%20Syst%20Biol&amp;volume=252&amp;issue=5&amp;pages=622-625&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31">Shadel G, Baldwin T: Identification of a distantly located regulatory element in the luxD gene required for negative autoregulation of the Vibrio fischeri luxR gene. J Biol Chem. 1992, 267 (11): 7690-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1560004"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20distantly%20located%20regulatory%20element%20in%20the%20luxD%20gene%20required%20for%20negative%20autoregulation%20of%20the%20Vibrio%20fischeri%20luxR%20gene&amp;author=G.%20Shadel&amp;author=T.%20Baldwin&amp;journal=J%20Biol%20Chem&amp;volume=267&amp;issue=11&amp;pages=7690&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32">Gillespie D: Exact stochastic simulation of coupled chemical reactions. J Phys Chem. 1977, 81 (25): 2340-2361. 10.1021/j100540a008.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1021/j100540a008"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Exact%20stochastic%20simulation%20of%20coupled%20chemical%20reactions&amp;author=D.%20Gillespie&amp;journal=J%20Phys%20Chem&amp;volume=81&amp;issue=25&amp;pages=2340-2361&amp;publication_year=1977&amp;doi=10.1021%2Fj100540a008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33">Rosenfeld N, Young JW, Alon U, Swain PS, Elowitz MB: Gene regulation at the single-cell level. Sci (New York, N.Y.). 2005, 307 (5717): 1962-5. 10.1126/science.1106914.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1126/science.1106914"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Gene%20regulation%20at%20the%20single-cell%20level&amp;author=N.%20Rosenfeld&amp;author=JW.%20Young&amp;author=U.%20Alon&amp;author=PS.%20Swain&amp;author=MB.%20Elowitz&amp;journal=Sci%20%28New%20York%2C%20N.Y.%29&amp;volume=307&amp;issue=5717&amp;pages=1962-5&amp;publication_year=2005&amp;doi=10.1126%2Fscience.1106914"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34">Canela-Xandri O, SaguÃ©s F, Buceta J: Interplay between intrinsic noise and the stochasticity of the cell cycle in bacterial colonies. Biophys J. 2010, 98 (11): 2459-68. 10.1016/j.bpj.2010.02.045.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20513389"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2877355"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.bpj.2010.02.045"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Interplay%20between%20intrinsic%20noise%20and%20the%20stochasticity%20of%20the%20cell%20cycle%20in%20bacterial%20colonies&amp;author=O.%20Canela-Xandri&amp;author=F.%20Sagu%C3%A9s&amp;author=J.%20Buceta&amp;journal=Biophys%20J&amp;volume=98&amp;issue=11&amp;pages=2459-68&amp;publication_year=2010&amp;doi=10.1016%2Fj.bpj.2010.02.045"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35">Lu T, Volfson D, Tsimring L, Hasty J: Cellular growth and division in the Gillespie algorithm. Syst Biol (Stevenage). 2004, 1: 121-10.1049/sb:20045016.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1049/sb%3A20045016"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Cellular%20growth%20and%20division%20in%20the%20Gillespie%20algorithm&amp;author=T.%20Lu&amp;author=D.%20Volfson&amp;author=L.%20Tsimring&amp;author=J.%20Hasty&amp;journal=Syst%20Biol%20%28Stevenage%29&amp;volume=1&amp;pages=121&amp;publication_year=2004&amp;doi=10.1049%2Fsb%3A20045016"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36">KÃ¦rn M, Elston T, Blake W, Collins J: Stochasticity in gene expression: from theories to phenotypes. Nat Rev Genet. 2005, 6 (6): 451-464. 10.1038/nrg1615.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15883588"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nrg1615"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Stochasticity%20in%20gene%20expression%3A%20from%20theories%20to%20phenotypes&amp;author=M.%20K%C3%A6rn&amp;author=T.%20Elston&amp;author=W.%20Blake&amp;author=J.%20Collins&amp;journal=Nat%20Rev%20Genet&amp;volume=6&amp;issue=6&amp;pages=451-464&amp;publication_year=2005&amp;doi=10.1038%2Fnrg1615"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37">Yu J, Xiao J, Ren X, Lao K, Xie XS: Probing gene expression in live cells, one protein molecule at a time. Sci (New York, N.Y.). 2006, 311 (5767): 1600-3. 10.1126/science.1119623.<span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1126/science.1119623"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Probing%20gene%20expression%20in%20live%20cells%2C%20one%20protein%20molecule%20at%20a%20time&amp;author=J.%20Yu&amp;author=J.%20Xiao&amp;author=X.%20Ren&amp;author=K.%20Lao&amp;author=XS.%20Xie&amp;journal=Sci%20%28New%20York%2C%20N.Y.%29&amp;volume=311&amp;issue=5767&amp;pages=1600-3&amp;publication_year=2006&amp;doi=10.1126%2Fscience.1119623"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38">Cai L, Friedman N, Xie XS: Stochastic protein expression in individual cells at the single molecule level. Nature. 2006, 440 (7082): 358-62. 10.1038/nature04599.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16541077"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nature04599"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Stochastic%20protein%20expression%20in%20individual%20cells%20at%20the%20single%20molecule%20level&amp;author=L.%20Cai&amp;author=N.%20Friedman&amp;author=XS.%20Xie&amp;journal=Nature&amp;volume=440&amp;issue=7082&amp;pages=358-62&amp;publication_year=2006&amp;doi=10.1038%2Fnature04599"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39">Ozbudak EM, Thattai M, Kurtser I, Grossman AD, van Oudenaarden A: Regulation of noise in the expression of a single gene. Nat Genet. 2002, 31: 69-73. 10.1038/ng869.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11967532"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/ng869"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Regulation%20of%20noise%20in%20the%20expression%20of%20a%20single%20gene&amp;author=EM.%20Ozbudak&amp;author=M.%20Thattai&amp;author=I.%20Kurtser&amp;author=AD.%20Grossman&amp;author=A.%20van%20Oudenaarden&amp;journal=Nat%20Genet&amp;volume=31&amp;pages=69-73&amp;publication_year=2002&amp;doi=10.1038%2Fng869"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40">Bennett MR, Hasty J: Microfluidic devices for measuring gene network dynamics in single cells. Nat Rev Genet. 2009, 10 (9): 628-38. 10.1038/nrg2625.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19668248"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2931582"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nrg2625"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Microfluidic%20devices%20for%20measuring%20gene%20network%20dynamics%20in%20single%20cells&amp;author=MR.%20Bennett&amp;author=J.%20Hasty&amp;journal=Nat%20Rev%20Genet&amp;volume=10&amp;issue=9&amp;pages=628-38&amp;publication_year=2009&amp;doi=10.1038%2Fnrg2625"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41">Urbanowski M, Lostroh C, Greenberg E: Reversible acyl-homoserine lactone binding to purified Vibrio fischeri LuxR protein. J Bacteriol. 2004, 186 (3): 631-10.1128/JB.186.3.631-637.2004.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14729687"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC321501"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1128/JB.186.3.631-637.2004"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Reversible%20acyl-homoserine%20lactone%20binding%20to%20purified%20Vibrio%20fischeri%20LuxR%20protein&amp;author=M.%20Urbanowski&amp;author=C.%20Lostroh&amp;author=E.%20Greenberg&amp;journal=J%20Bacteriol&amp;volume=186&amp;issue=3&amp;pages=631&amp;publication_year=2004&amp;doi=10.1128%2FJB.186.3.631-637.2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42">Kaufmann G, Sartorio R, Lee S, Rogers C, Meijler M, Moss J, Clapham B, Brogan A, Dickerson T, Janda K: Revisiting quorum sensing: discovery of additional chemical and biological functions for 3-oxo-N-acylhomoserine lactones. Proc Natl Acad Sci U S A. 2005, 102 (2): 309-10.1073/pnas.0408639102.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15623555"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC544315"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1073/pnas.0408639102"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Revisiting%20quorum%20sensing%3A%20discovery%20of%20additional%20chemical%20and%20biological%20functions%20for%203-oxo-N-acylhomoserine%20lactones&amp;author=G.%20Kaufmann&amp;author=R.%20Sartorio&amp;author=S.%20Lee&amp;author=C.%20Rogers&amp;author=M.%20Meijler&amp;author=J.%20Moss&amp;author=B.%20Clapham&amp;author=A.%20Brogan&amp;author=T.%20Dickerson&amp;author=K.%20Janda&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=102&amp;issue=2&amp;pages=309&amp;publication_year=2005&amp;doi=10.1073%2Fpnas.0408639102"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43">Roberts C, Anderson KL, Murphy E, Projan SJ, Mounts W, Hurlburt B, Smeltzer M, Overbeek R, Disz T, Dunman PM: Characterizing the effect of the Staphylococcus aureus virulence factor regulator, SarA, on log-phase mRNA half-lives. J Bacteriol. 2006, 188 (7): 2593-603. 10.1128/JB.188.7.2593-2603.2006.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16547047"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1428411"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1128/JB.188.7.2593-2603.2006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Characterizing%20the%20effect%20of%20the%20Staphylococcus%20aureus%20virulence%20factor%20regulator%2C%20SarA%2C%20on%20log-phase%20mRNA%20half-lives&amp;author=C.%20Roberts&amp;author=KL.%20Anderson&amp;author=E.%20Murphy&amp;author=SJ.%20Projan&amp;author=W.%20Mounts&amp;author=B.%20Hurlburt&amp;author=M.%20Smeltzer&amp;author=R.%20Overbeek&amp;author=T.%20Disz&amp;author=PM.%20Dunman&amp;journal=J%20Bacteriol&amp;volume=188&amp;issue=7&amp;pages=2593-603&amp;publication_year=2006&amp;doi=10.1128%2FJB.188.7.2593-2603.2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44">Kaplan H, Greenberg E: Diffusion of autoinducer is involved in regulation of the Vibrio fischeri luminescence system. J Bacteriol. 1985, 163 (3): 1210-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3897188"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC219261"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Diffusion%20of%20autoinducer%20is%20involved%20in%20regulation%20of%20the%20Vibrio%20fischeri%20luminescence%20system&amp;author=H.%20Kaplan&amp;author=E.%20Greenberg&amp;journal=J%20Bacteriol&amp;volume=163&amp;issue=3&amp;pages=1210&amp;publication_year=1985"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45">Reshes G, Vanounou S, Fishov I, Feingold M: Timing the start of division in E. coli: a single-cell study. Phys Biol. 2008, 5 (4): 046001-10.1088/1478-3975/5/4/046001.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18997273"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1088/1478-3975/5/4/046001"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Timing%20the%20start%20of%20division%20in%20E.%20coli%3A%20a%20single-cell%20study&amp;author=G.%20Reshes&amp;author=S.%20Vanounou&amp;author=I.%20Fishov&amp;author=M.%20Feingold&amp;journal=Phys%20Biol&amp;volume=5&amp;issue=4&amp;pages=046001&amp;publication_year=2008&amp;doi=10.1088%2F1478-3975%2F5%2F4%2F046001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46">Trueba F, Koppes L: Exponential growth of Escherichia coli B/r during its division cycle is demonstrated by the size distribution in liquid culture. Arch Microbiol. 1998, 169: 491-496. 10.1007/s002030050601.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9575234"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1007/s002030050601"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Exponential%20growth%20of%20Escherichia%20coli%20B%2Fr%20during%20its%20division%20cycle%20is%20demonstrated%20by%20the%20size%20distribution%20in%20liquid%20culture&amp;author=F.%20Trueba&amp;author=L.%20Koppes&amp;journal=Arch%20Microbiol&amp;volume=169&amp;pages=491-496&amp;publication_year=1998&amp;doi=10.1007%2Fs002030050601"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47">Hong D, Saidel WM, Man S, Martin JV: Extracellular noise-induced stochastic synchronization in heterogeneous quorum sensing network. J theor biol. 2007, 245 (4): 726-736. 10.1016/j.jtbi.2006.12.006.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17239902"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.jtbi.2006.12.006"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Extracellular%20noise-induced%20stochastic%20synchronization%20in%20heterogeneous%20quorum%20sensing%20network&amp;author=D.%20Hong&amp;author=WM.%20Saidel&amp;author=S.%20Man&amp;author=JV.%20Martin&amp;journal=J%20theor%20biol&amp;volume=245&amp;issue=4&amp;pages=726-736&amp;publication_year=2007&amp;doi=10.1016%2Fj.jtbi.2006.12.006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48">Zhou T, Chen L, Aihara K: Molecular communication through stochastic synchronization induced by extracellular fluctuations. Phys rev lett. 2005, 95 (17): 178103-<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16383875"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1103/PhysRevLett.95.178103"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Molecular%20communication%20through%20stochastic%20synchronization%20induced%20by%20extracellular%20fluctuations&amp;author=T.%20Zhou&amp;author=L.%20Chen&amp;author=K.%20Aihara&amp;journal=Phys%20rev%20lett&amp;volume=95&amp;issue=17&amp;pages=178103&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49">Horsthemke W, Lefever R: Noise-induced transitions: theory and applications in physics, chemistry and biology. 1984, Springer-Verlag, Berlin Heidelberg<span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Noise-induced%20transitions%3A%20theory%20and%20applications%20in%20physics%2C%20chemistry%20and%20biology&amp;author=W.%20Horsthemke&amp;author=R.%20Lefever&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR50">PÃ©rez PD, Weiss JT, Hagen SJ: Noise and crosstalk in two quorum-sensing inputs of Vibrio fischeri. BMC Syst Biol. 2011, 5: 153-10.1186/1752-0509-5-153.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21959018"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3224347"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1186/1752-0509-5-153"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Noise%20and%20crosstalk%20in%20two%20quorum-sensing%20inputs%20of%20Vibrio%20fischeri&amp;author=PD.%20P%C3%A9rez&amp;author=JT.%20Weiss&amp;author=SJ.%20Hagen&amp;journal=BMC%20Syst%20Biol&amp;volume=5&amp;pages=153&amp;publication_year=2011&amp;doi=10.1186%2F1752-0509-5-153"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR51">Teng SW, Wang Y, Tu KC, Long T, Mehta P, Wingreen NS, Bassler BL, Ong NP: Measurement of the copy number of the master quorum-sensing regulator of a bacterial cell. Biophys J. 2010, 98 (9): 2024-31. 10.1016/j.bpj.2010.01.031.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20441767"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2862190"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.bpj.2010.01.031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Measurement%20of%20the%20copy%20number%20of%20the%20master%20quorum-sensing%20regulator%20of%20a%20bacterial%20cell&amp;author=SW.%20Teng&amp;author=Y.%20Wang&amp;author=KC.%20Tu&amp;author=T.%20Long&amp;author=P.%20Mehta&amp;author=NS.%20Wingreen&amp;author=BL.%20Bassler&amp;author=NP.%20Ong&amp;journal=Biophys%20J&amp;volume=98&amp;issue=9&amp;pages=2024-31&amp;publication_year=2010&amp;doi=10.1016%2Fj.bpj.2010.01.031"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR52">Tu KC, Long T, Svenningsen SL, Wingreen NS, Bassler BL: Negative feedback loops involving small regulatory RNAs precisely control the Vibrio harveyi quorum-sensing response. Mol cell. 2010, 37 (4): 567-79. 10.1016/j.molcel.2010.01.022.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20188674"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844700"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.molcel.2010.01.022"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Negative%20feedback%20loops%20involving%20small%20regulatory%20RNAs%20precisely%20control%20the%20Vibrio%20harveyi%20quorum-sensing%20response&amp;author=KC.%20Tu&amp;author=T.%20Long&amp;author=SL.%20Svenningsen&amp;author=NS.%20Wingreen&amp;author=BL.%20Bassler&amp;journal=Mol%20cell&amp;volume=37&amp;issue=4&amp;pages=567-79&amp;publication_year=2010&amp;doi=10.1016%2Fj.molcel.2010.01.022"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR53">BalÃ¡zsi G, van Oudenaarden, Collins JJ: Cellular decision making and biological noise: from microbes to mammals. Cell. 2011, 144 (6): 910-25. 10.1016/j.cell.2011.01.030.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21414483"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3068611"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1016/j.cell.2011.01.030"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Cellular%20decision%20making%20and%20biological%20noise%3A%20from%20microbes%20to%20mammals&amp;author=G.%20Bal%C3%A1zsi&amp;author=A.%20van%20Oudenaarden&amp;author=JJ.%20Collins&amp;journal=Cell&amp;volume=144&amp;issue=6&amp;pages=910-25&amp;publication_year=2011&amp;doi=10.1016%2Fj.cell.2011.01.030"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR54">Acar M, Becskei A, van Oudenaarden: Enhancement of cellular memory by reducing stochastic transitions. Nature. 2005, 435 (7039): 228-32. 10.1038/nature03524.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15889097"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1038/nature03524"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Enhancement%20of%20cellular%20memory%20by%20reducing%20stochastic%20transitions&amp;author=M.%20Acar&amp;author=A.%20Becskei&amp;author=A.%20van%20Oudenaarden&amp;journal=Nature&amp;volume=435&amp;issue=7039&amp;pages=228-32&amp;publication_year=2005&amp;doi=10.1038%2Fnature03524"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR55">NenÃ© NR, GarcÃ­a-Ojalvo J, Zaikin A: Speed-dependent cellular decision making in nonequilibrium genetic circuits. PLoS One. 2012, 7 (3): e32779-10.1371/journal.pone.0032779.<span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22427883"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrencePMCID"><a target="_blank" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3302799"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" href="http://dx.doi.org/10.1371/journal.pone.0032779"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" href="http://scholar.google.com/scholar_lookup?title=Speed-dependent%20cellular%20decision%20making%20in%20nonequilibrium%20genetic%20circuits&amp;author=NR.%20Nen%C3%A9&amp;author=J.%20Garc%C3%ADa-Ojalvo&amp;author=A.%20Zaikin&amp;journal=PLoS%20One&amp;volume=7&amp;issue=3&amp;pages=e32779&amp;publication_year=2012&amp;doi=10.1371%2Fjournal.pone.0032779"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection">Copyright</h2><div class="js-CollapseSection"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">Â©&nbsp;Weber and Buceta; licensee BioMed Central Ltd.&nbsp;2013<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published under license to BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                            


                        </div>
                    </div>
                    <div class="main__sidebar main__sidebar--right grid__col grid__col--xs-12 grid__col--md-3" id="main-sidebar">
                        
                        <div class="block">
                            <aside role="complementary">
                                <div class="SideBox">
                                    
    <p class="u-marginBtmM">
        <a class="button button--size-300 button--priority-1 button--full" id="articlePdf" target="_blank" href="/track/pdf/10.1186/1752-0509-7-6?site&#x3D;bmcsystbiol.biomedcentral.com" data-track-event="click" data-event-category="PDF Download"
           data-event-action="Click download article PDF"
           data-event-label="10.1186/1752-0509-7-6">Download PDF</a>
    </p>

                                    
                                    
<div class="Collapse Collapse--button" id="js-exportCitations">
    <h3 class="Collapse_toggle Collapse_toggle--right" data-toggle="collapse" data-target="js-exportCitations">Export citations</h3>
    <div class="Panel Panel--s Collapse_content">
        <div class="Panel_section">
            <h4 class="Panel_sectionTitle">Citations & References</h4>
            <ul class="ListDownload" id="articleReferences">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references"
                       data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=refman&flavour=full">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a>
                </li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote citations and references" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=endnote&flavour=full">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef citations and references" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=bibtex&flavour=full">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
        <div class="Panel_section">
            <h4 class="Panel_sectionTitle">Article citation</h4>
            <ul class="ListDownload">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=refman&flavour=citation">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote article citations" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=endnote&flavour=citation">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef article citations" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=bibtex&flavour=citation">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
        <div class="Panel_section">
            <h4 class="Panel_sectionTitle">References</h4>
            <ul class="ListDownload">
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=refman&flavour=references">Papers, Zotero, Reference Manager, RefWorks <span>(.RIS)</span></a>
                </li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download EndNote references" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=endnote&flavour=references">EndNote <span>(.ENW)</span></a></li>
                <li><a data-track-event="click" data-event-category="Article Page" data-event-action="Download Mendeley, JabRef references" data-event-label="10.1186/1752-0509-7-6"
                       href="http://citation-needed.services.springer.com/v2/references/10.1186/1752-0509-7-6?format=bibtex&flavour=references">Mendeley, JabRef <span>(.BIB)</span></a></li>
            </ul>
        </div>
    </div>
</div>

                                </div>
                                <div class="SideBox">
                                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" class="table-of-contents" aria-label="table of contents"><div class="table-of-contents__content" data-sticky="toc"><h3 class="table-of-contents__title is-active" data-toggle="collapse" data-target="tocMenu" data-event-category="Table of Contents" data-track-event="click" data-event-action="Toggle ToC" data-event-label="Toggle">Table of Contents</h3><ul class="table-of-contents__menu is-open" role="menu" id="tocMenu"><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Abs1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Abstract">Abstract</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Background">Background</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec2" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Methods">Methods</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec9" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Results">Results</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec13" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Discussion">Discussion</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Sec14" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Conclusions">Conclusions</a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Declarations" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Declarations">
                        Declarations
                      </a></li><li role="menuitem" class="table-of-contents__item"><a class="table-of-contents__link" href="#Bib1" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="References">References</a></li><li role="menuitem" class="table-of-contents__item is-hidden"><a class="table-of-contents__link" href="#comments" data-event-category="Table of Contents" data-track-event="click" data-event-action="Link Clicked" data-event-label="Comments">Comments</a></li></ul></div></nav>
                                </div>
                                <div class="SideBox">
                                    <h3 class="SideBox_title">Metrics</h3>
                                    <ul class="SideBox_list list-stacked list-stacked--small list-stacked--2">
                                        <div id="article-access-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/access-count/10.1186/1752-0509-7-6 #accessCount-container"></div>
                                        <div id="citation-count-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/citation-count/10.1186/1752-0509-7-6 #count-container"></div>
                                        <div id="altmetric-loader" class="placeholder" data-replace="true" data-placeholder="/placeholder/v1/article/altmetric/10.1186/1752-0509-7-6 #altmetric-container"></div>
                                    </ul>
                                </div>
                                
    <div id="sociaMediaShare" class="SideBox">
        <h3 class="SideBox_title">Share this article</h3>
        <ul id="shareButtons" class="share">
            <li class="share__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter share__button share__button--twitter" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Twitter" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on Twitter</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook share__button share__button--facebook" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Facebook" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on Facebook</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn share__button share__button--linkedin" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share LinkedIn" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on LinkedIn</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo share__button share__button--weibo" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Weibo" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on Weibo</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus share__button share__button--gplus" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Google+" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on Google Plus</span>
                </button>
            </li>
            <li class="share__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit share__button share__button--reddit" data-track-event="click"
                        data-event-category="Article Page" data-event-action="Share Reddit" data-event-label="10.1186/1752-0509-7-6">
                    <span class="u-srOnly">Share on Reddit</span>
                </button>
            </li>
        </ul>
    </div>

                                
    

                                
    
    <div id="articleSections" class="SideBox">
        <h3 class="SideBox_title">Section</h3>
    <ul class="list-stacked list-stacked--2 list-stacked--small">
        
        <li class="list-stacked__item">
            <a class="list-stacked__link" href="/articles/sections/networks-and-information-flow"
                   data-track-event="click"
                   data-event-category="Article Page"
                   data-event-action="Click section link"
                   data-event-label="10.1186/1752-0509-7-6">Networks and information flow</a>
            </li>
    
        </ul>
    </div>
    

                                
    

                                <div class="SideBox">
                                    <h3 class="SideBox_title">Other Actions</h3>
                                    <ul class="list-stacked list-stacked--2 list-stacked--small">
                                        <li class="list-stacked__item">
                                            
    <a data-track-event="click" data-event-category="Article Page" data-event-action="Order article reprints" data-event-label="10.1186/1752-0509-7-6" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186%2F1752-0509-7-6&volume=7&issue=1&title=Dynamics+of+the+quorum+sensing+switch%3A+stochastic+and+non-stationary+effects&author_name=Marc+Weber&start_page=1&end_page=15">Order reprint</a>

                                        </li>
                                    </ul>
                                </div>
                                
        
        <div id="sociaMediaAccounts" class="SideBox">
            <h3 class="SideBox_title">Follow</h3>
            <ul class="follow follow--inline">
                
                <li class="follow__item">
                    <a class="follow__link follow__link--twitter-2" href="https://twitter.com/BMC_series" data-track-event="click" data-event-category="Social Media"
                       data-event-action="Click Follow Social Media" data-event-label="Twitter">
                        <span class="follow__label">Follow us on Twitter</span>
                    </a>
                </li>
                

                

                
            </ul>
        </div>
        
    
                            </aside>
                            
    <div id="FulltextAd" class="adsbox Ad Ad--skyscraper">
        <div class="Ad_content">
            <p class="Ad_label">Advertisement</p>
            <span id="pbgrd-43415139"></span>
        </div>
    </div>

                        </div>
                    </div>
                </div>
            </div>
        </main>
        <p style="display: none;" class="footer-dynamic-legal">
    <a class="footer-dynamic-legal__link" title="View Legal Information" href="http://www.miibeian.gov.cn">
        &#27818;ICP&#22791;15051854&#21495;-3
    </a>
</p>
        
            <div class="journal-footer">
                <div class="journal-footer__inner">
                    <div class="journal-footer__summary">
                        <h4 class="journal-footer__title">
                            BMC Systems Biology
                        </h4>
                        <p class="journal-footer__issn">ISSN: 1752-0509</p>
                    </div>
                    
                        <div class="journal-footer__contact">
                            <h4 class="journal-footer__contact-title">Contact us</h4>
                            <ul class="journal-footer__contact-list">
                                
                                    <li class="journal-footer__contact-item">Editorial email: <a href="mailto:bmcsystbiol@biomedcentral.com">bmcsystbiol@biomedcentral.com</a></li>
                                
                                
                                    <li class="journal-footer__contact-item">Support email: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
         
            <footer>
                <div class="footer">
                    <div class="footer__header">
                        <img srcset="/static/images/biomed-central-logo-white.d75eb464.svg" src="/static/images/biomed-central-logo-white.c1ec99e0.png" class="logo__img footer__logo" alt="Biomed Central" />
                        
                            <h2 class="footer__menu-title">Publisher main menu</h2>
                            <ul class="footer__primary-menu">
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/journals">Explore journals</a>
                                    </li>
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/getpublished">Get published</a>
                                    </li>
                                
                                    <li class="footer__primary-item">
                                        <a class="footer__link footer__primary-link" href="//www.biomedcentral.com/about">About BioMed Central</a>
                                    </li>
                                
                            </ul>
                        
                    </div>
                    <div id="Test-FooterExists" class="footer__content">
                        <div class="footer__legal">
                            <p class="footer__terms">By continuing to use this website, you agree to our <a class="footer__link footer__link--inline" href="//www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>, <a class="footer__link footer__link--inline"
                                    href="//www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy
                                statement</a> and <a class="footer__link footer__link--inline" href="//www.biomedcentral.com/about/cookies">Cookies</a> policy.</p>
                        </div>
                        
    <nav id="publisherFooter" class="footer__nav">
        <h3 class="footer__menu-title">Publisher secondary menu</h3>
        <div class="footer__nav-inner">
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/contacts">Contact us</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/jobs">Jobs</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/logon?url&#x3D;/my/manuscripts">Manage manuscripts</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/logon?url&#x3D;/my/preferences">Sign up for article alerts</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/logon?url&#x3D;/my/preferences">Manage article alerts</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="https://biomedcentral.typeform.com/to/ve8NOd">Leave feedback</a>
                        
                    </li>
                    
                </ul>
            </div>
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/press-centre">Press center</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                        
                    </li>
                    
                </ul>
            </div>
            
            <div class="footer__menu">
                <ul class="footer__list">
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/policies">Policies</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/licensing">Licensing</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/terms-and-conditions">Terms and conditions</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/terms-and-conditions/privacy-statement">Privacy statement</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/accessibility">Accessibility</a>
                        
                    </li>
                    
                    <li class="footer__item">
                        <a class="footer__link" href="//www.biomedcentral.com/about/cookies">Cookies</a>
                        
                    </li>
                    
                </ul>
            </div>
            

            <div class="footer__menu footer__menu--follow">
                <div class="footer__follow">
                    <h3 class="footer__follow-title">Follow BioMed Central</h3>
                    <ul class="follow follow--inline">
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--twitter" href="https://twitter.com/biomedcentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Twitter">
                                <span class="follow__label">Twitter</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--facebook" href="https://www.facebook.com/BioMedCentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Facebook">
                                <span class="follow__label">Facebook</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--gplus" href="https://plus.google.com/106226337984152901734/" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media"
                               data-event-label="Google Plus">
                                <span class="follow__label">Google Plus</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--youtube" href="https://www.youtube.com/user/BioMedCentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="YouTube">
                                <span class="follow__label">YouTube</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--linkedin" href="https://www.linkedin.com/company/biomed-central" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="LinkedIn">
                                <span class="follow__label">LinkedIn</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--reddit" href="https://www.reddit.com/domain/biomedcentral.com" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Reddit">
                                <span class="follow__label">Reddit</span>
                            </a>
                        </li>
                        
                        
                        <li class="footer__follow-item follow__item">
                            <a class="follow__link follow__link--weibo" href="http://www.weibo.com/biomedcentral" target="_blank" data-track-event="click" data-event-category="Social Media" data-event-action="Click Follow Social Media" data-event-label="Weibo">
                                <span class="follow__label">Weibo</span>
                            </a>
                        </li>
                        
                    </ul>
                </div>
            </div>
        </div>
    </nav>


                    </div>
                </div>
                <div class="branding-bar">
                    <div class="logo branding-bar__logo">
                        <img srcset="/static/images/logo-springernature.06a51479.svg" src="/static/images/logo-springernature.ecd3ff3f.png" class="logo__img branding-bar-logo__img" alt="Springer Nature Logo" />
                    </div>
                    <p class="branding-bar__copyright">&#169; 2016 BioMed Central Ltd unless otherwise stated. Part of <a class="branding-bar__link" href="http://www.springernature.com">Springer Nature</a>.</p>
                </div>
            </footer>
        
    </div>
    
    <img class='tracker' style='display:none' src='/track/article/10.1186/1752-0509-7-6' alt=""/>

    <div id="JFulltext"></div>

    
    <script type="text/javascript" src="//cdn.pbgrd.com/bmc.js" async="async"></script>

    

    <script async="true" src="/static/js/oscar.75339aff.js"></script>




<script type="text/javascript">
    if (config) {
        (function (d) {
            Array.prototype.forEach.call(d.getElementsByClassName('RealMediaAd'), function (adElement) {
                var iframe = d.createElement('iframe');
                iframe.frameBorder = '0';
                iframe.scrolling = 'no';
                var position = adElement.getAttribute("data-position"),
                        iframeInDoc = adElement.getElementsByClassName('Ad_content')[0].appendChild(iframe),
                        doc = iframeInDoc.contentWindow.document;

                doc.open().write('<body onload="window.location.href = '
                        + '\'/placeholder/v1/advertising/realMedia?siteWithPath=' + encodeURIComponent(config.siteWithPath)
                        + '&placement=' + position + '\';">');
                doc.close();
            });
        })(document);
    }
</script>



    
        <script type="text/javascript">
            document.addEventListener('readystatechange', function () {
                if (document.readyState === 'complete') {
                    setTimeout(function () {
                        var script = document.createElement("script");
                        script.type = "text/javascript";
                        script.src = "https://recommendations.springernature.com/latest/entry-point.js";
                        document.getElementsByTagName("head")[0].appendChild(script);
                    }, 0);
                }
            });
        </script>
    


    <script type="text/javascript">
        var _kiq = _kiq || [];
        (function () {
            setTimeout(function () {
                var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script');
                s.type = 'text/javascript';
                s.async = true;
                s.src = '//s3.amazonaws.com/ki.js/47412/9CC.js';
                f.parentNode.insertBefore(s, f);
            }, 1);
            setTimeout(function () {
                var d = document, f = d.getElementsByTagName('script')[0], s = d.createElement('script');
                s.type = 'text/javascript';
                s.async = true;
                s.src = '//s3.amazonaws.com/ki.js/47412/aC6.js';
                f.parentNode.insertBefore(s, f);
            }, 1);
        })();
    </script>

</body>
</html>
